{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "a3ce03cc",
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy\n",
    "import pandas as pd\n",
    "import nltk\n",
    "from nltk.stem.porter import *"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "337441e6",
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.set_option(\"display.max_rows\", None, \"display.max_columns\", None)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c3532a85",
   "metadata": {},
   "source": [
    "#  Lemmatizing Keywords"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "id": "01fde60a",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Read in the CSV to lemmatize the keywords\n",
    "df_keywords = pd.read_csv(\"words2stem_lemmatize.csv\", sep=\"\\t\") # sep=\"\\t\" weil in Zeile 147 Kommas in der Zelle stehen"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "b4e5f0ec",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Remove colons\n",
    "df_keywords = df_keywords.replace('\\:','', regex=True)\n",
    "# Rename column\n",
    "df_keywords = df_keywords.rename(columns={'DOI': 'keywords'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "9b9cf088",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "D:\\TH_Koeln\\WS21-22\\Bachelorarbeit\\python_scripts\\venv_spacy\\lib\\site-packages\\torch\\autocast_mode.py:162: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling\n",
      "  warnings.warn('User provided device_type of \\'cuda\\', but CUDA is not available. Disabling')\n"
     ]
    }
   ],
   "source": [
    "nlp = spacy.load(\"en_core_web_trf\")\n",
    "# x ist der text der Spalte keywords, nlp(x) erstellt daraus ein Dokument und y repräsentoert jeden Token, welcher mit y.lemma_ lemmatisiert wird \n",
    "# \" \".join(...) fügt alle gefundenen Lemmata zu einer einzigen durch Leerzeichen getrennten Zeichenfolge zusammen.\n",
    "df_keywords[\"lemmatized\"] = df_keywords['keywords'].apply(lambda x: \" \".join([y.lemma_ for y in nlp(x)]))\n",
    "# https://stackoverflow.com/questions/62712963/using-spacy-to-lemmatize-a-column-of-parsed-html-text-in-a-pandas-dataframe"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "id": "e2150082",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Lowercase letters that were not lowercased during lemmatization\n",
    "df_keywords[\"lemmatized\"] = df_keywords[\"lemmatized\"].str.lower() "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "76ba4d8d",
   "metadata": {},
   "source": [
    "# Stemming Keywords"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "046b240a",
   "metadata": {},
   "outputs": [],
   "source": [
    "stemmer = PorterStemmer()\n",
    "keywords_stemmed = []\n",
    "\n",
    "for words in df_keywords['keywords']:\n",
    "    \n",
    "    # if keyword consists of more then one words, split them, stem them, an put them back together to one string\n",
    "    if \" \" in words:\n",
    "        words_split = words.split() \n",
    "        words_stemmed = [stemmer.stem(x) for x in words_split]\n",
    "        words_stemmed = ' '.join(map(str, words_stemmed))\n",
    "        keywords_stemmed.append(words_stemmed)\n",
    "        \n",
    "    # if keyword consists of one word just stem it\n",
    "    else:\n",
    "        keywords_stemmed.append(stemmer.stem(words))\n",
    "        \n",
    "df_keywords[\"stemmed\"] = keywords_stemmed"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "id": "5c3ecde6",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Saving the lemmatized terms to CSV\n",
    "df_keywords.to_csv('keywords_lemmatized_stemmed.csv', index=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "15adb1ef",
   "metadata": {},
   "source": [
    "# SpaCy Rule-based entity recognition"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "3a66484f",
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy\n",
    "from spacy.lang.en import English\n",
    "from spacy.pipeline import EntityRuler"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "47f4e360",
   "metadata": {},
   "source": [
    "## Expand species names list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "723325b1",
   "metadata": {},
   "outputs": [],
   "source": [
    "# creates lists for NER model\n",
    "df_keywords = pd.read_csv(\"keywords_lemmatized_stemmed.csv\")\n",
    "\n",
    "# remove line 0 to 10 because they are unnecessary\n",
    "df_keywords = df_keywords.drop(range(0,11))\n",
    "\n",
    "list_keywords = df_keywords[\"keywords\"].tolist()\n",
    "\n",
    "df_species = pd.read_csv(\"species_names_list.csv\")\n",
    "list_species = df_species[\"species\"].tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "id": "7cd3a678",
   "metadata": {},
   "outputs": [],
   "source": [
    "# lowercase all species names\n",
    "species_low = [name.lower() for name in list_species]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "id": "cc24581e",
   "metadata": {},
   "outputs": [],
   "source": [
    "# separates all double names into single words, lower case them and adds them to list\n",
    "species_split = []\n",
    "for name in list_species:\n",
    "    if \" \" in name:\n",
    "        name_split = name.split()\n",
    "        for x in name_split:\n",
    "            x = x.strip()\n",
    "            x = x.lower() \n",
    "            species_split.append(x)\n",
    "\n",
    "# removes duplicates\n",
    "species_split = list(dict.fromkeys(species_split))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "id": "c6c73b5b",
   "metadata": {},
   "outputs": [],
   "source": [
    "# capitalizes all names\n",
    "species_cap = [name.capitalize() for name in species_expanded]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "id": "3eed3750",
   "metadata": {},
   "outputs": [],
   "source": [
    "# merges all species lists an removes duplicates\n",
    "species_final = list_species + species_low + species_split + species_cap\n",
    "species_final = list(dict.fromkeys(species_final))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2e3d2522",
   "metadata": {},
   "source": [
    "## Species and Keywords NER Model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "6312221c",
   "metadata": {},
   "outputs": [],
   "source": [
    "# creates the pattern for ner keywords rules\n",
    "def create_keywords_ner_pattern():\n",
    "    species = species_final\n",
    "    keywords = list_keywords\n",
    "    patterns = []\n",
    "    for item in species:\n",
    "        pattern = {\n",
    "            \"label\": \"SPECIES\",\n",
    "            \"pattern\": item\n",
    "        }\n",
    "        patterns.append(pattern)\n",
    "    for item in keywords:\n",
    "        pattern = {\n",
    "            \"label\": \"KEYWORD\",\n",
    "            \"pattern\": item\n",
    "        }\n",
    "        patterns.append(pattern)\n",
    "        \n",
    "    return (patterns)\n",
    "\n",
    "# create rules for ner keywords model and creates the model\n",
    "def generate_rules(patterns):\n",
    "    nlp = English()\n",
    "    source_nlp = spacy.load(\"en_core_web_trf\")\n",
    "    nlp.add_pipe(\"ner\", source=source_nlp)\n",
    "    ruler = EntityRuler(nlp)\n",
    "    ruler.add_patterns(patterns)\n",
    "    nlp.to_disk(\"keywords_ner\")\n",
    "\n",
    "# run the function    \n",
    "patterns = create_keywords_ner_pattern()\n",
    "#generate_rules(patterns)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "eb2591c3",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Konvertiere Spalte zu String, um als nlp-objekt erkannt zu werden\n",
    "df = pd.read_csv(\"fulltext_articles.csv\")\n",
    "list_ft = df.loc[:5, \"fulltext\"].tolist()\n",
    "string_ft = ' '.join(map(str, list_ft))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "7253b3a2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('Aliidiomarina taiwanensis', 'SPECIES'), ('Aliidiomarina haloalkalitolerans', 'SPECIES'), ('Aliidiomarina shirensis', 'SPECIES'), ('Aliidiomarina minuta', 'SPECIES'), ('Aliidiomarina iranensis', 'SPECIES'), ('Aliidiomarina soli', 'SPECIES'), ('Aliidiomarina maris', 'SPECIES'), ('Aliidiomarina celeris', 'SPECIES'), ('Aliidiomarina sedimenti', 'SPECIES'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('Escherichia coli', 'SPECIES'), ('species', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('Motility', 'KEYWORD'), ('temperature range', 'KEYWORD'), ('pH', 'KEYWORD'), ('concentration', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('concentration', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('production', 'KEYWORD'), ('production', 'KEYWORD'), ('pH', 'KEYWORD'), ('incubation period', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('species', 'KEYWORD'), ('type', 'KEYWORD'), ('motility', 'KEYWORD'), ('pH', 'KEYWORD'), ('concentration', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('description', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('salt', 'KEYWORD'), ('salt', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('concentration', 'KEYWORD'), ('methyl red', 'KEYWORD'), ('type', 'KEYWORD'), ('type', 'KEYWORD'), ('type', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('Mameliella alba', 'SPECIES'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('Mameliella alba', 'SPECIES'), ('type', 'KEYWORD'), ('genus', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('ability', 'KEYWORD'), ('genus', 'KEYWORD'), ('genus', 'KEYWORD'), ('type', 'KEYWORD'), ('Mameliella alba', 'SPECIES'), ('Antarctobacter jejuensis', 'SPECIES'), ('Antarctobacter heliothermus', 'SPECIES'), ('Maliponia aquimaris', 'SPECIES'), ('Ponticoccus litoralis', 'SPECIES'), ('Aquicoccus porphyridii', 'SPECIES'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('motility', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('activity', 'KEYWORD'), ('activity', 'KEYWORD'), ('production', 'KEYWORD'), ('salt', 'KEYWORD'), ('production', 'KEYWORD'), ('methyl red', 'KEYWORD'), ('production', 'KEYWORD'), ('type', 'KEYWORD'), ('production', 'KEYWORD'), ('pH', 'KEYWORD'), ('species', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('pH', 'KEYWORD'), ('pH', 'KEYWORD'), ('methyl red', 'KEYWORD'), ('production', 'KEYWORD'), ('production', 'KEYWORD'), ('type', 'KEYWORD'), ('type', 'KEYWORD'), ('type', 'KEYWORD'), ('species', 'KEYWORD'), ('genus', 'KEYWORD'), ('type', 'KEYWORD')]\n"
     ]
    }
   ],
   "source": [
    "# Versuch 2 ERFOLGREICH!\n",
    "nlp = English()\n",
    "ruler = nlp.add_pipe(\"entity_ruler\")\n",
    "ruler.add_patterns(patterns)\n",
    "doc = nlp(string_ft)\n",
    "print([(ent.text, ent.label_) for ent in doc.ents])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a3b4be88",
   "metadata": {},
   "outputs": [],
   "source": [
    "# # convert list to string for lemmatizing\n",
    "#abstract_string = ' '.join(map(str, abstract_list))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "7c910060",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "genus KEYWORD\n",
      "genus KEYWORD\n",
      "genus KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "genus KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "Aliidiomarina taiwanensis SPECIES\n",
      "Aliidiomarina haloalkalitolerans SPECIES\n",
      "Aliidiomarina shirensis SPECIES\n",
      "Aliidiomarina minuta SPECIES\n",
      "Aliidiomarina iranensis SPECIES\n",
      "Aliidiomarina soli SPECIES\n",
      "Aliidiomarina maris SPECIES\n",
      "Aliidiomarina celeris SPECIES\n",
      "Aliidiomarina sedimenti SPECIES\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "pH KEYWORD\n",
      "pH KEYWORD\n",
      "Escherichia coli SPECIES\n",
      "species KEYWORD\n",
      "type KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "type KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "species KEYWORD\n",
      "species KEYWORD\n",
      "species KEYWORD\n",
      "species KEYWORD\n",
      "species KEYWORD\n",
      "species KEYWORD\n",
      "genus KEYWORD\n",
      "type KEYWORD\n"
     ]
    }
   ],
   "source": [
    "for ent in doc.ents[:50]:\n",
    "    print(ent.text,ent.label_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "eed8b784",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<span class=\"tex2jax_ignore\"><div class=\"entities\" style=\"line-height: 2.5; direction: ltr\">ANI, average nucleotide identity; ANIb, average nucleotide identity calculated using blast; ANIm, average nucleotide identity calculated using MUMmer; dDDH, digital DNA–DNA hybridization; dDDH, digital DNA–DNA hybridization.,The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina belongs to the family Idiomarinaceae, order Alteromonadales, class Gammaproteobacteria [1]. This \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " has two sister genera named Idiomarina and Pseudidiomarina in the List of Prokaryotic names with Standing in Nomenclature. Despite the reclassification of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Pseudidiomarina into the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Idiomarina, the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of both genera could not be distinguished from each other using phenotypic or chemotaxonomic characteristics [2], and recently the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Pseudidiomarina was reinstated following the results of genome-based analysis [3]. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina was close to the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Pseudidiomarina in the phylogenetic tree based on the 16S rRNA genes. However, it is distant from both sister genera in the phylogenomic tree based on the 78 core genes [3]. At the time of writing, a total of ten \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of this \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " were validly published, including \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina taiwanensis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [1], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina haloalkalitolerans\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [4], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina shirensis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [5], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina minuta\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [6], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina iranensis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [7], Aliidiomarina sanyensis [8], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina soli\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [9], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina maris\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [10], \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina celeris\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [11] and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aliidiomarina sedimenti\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " [12]. These \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " were isolated from a wide variety of habitats, including seawater [1, 4], saline lakes [5, 6], coastal-marine wetland [7], Spirulina platensis cultivation pool [8], soil [9] and sediments [10–12]. Members of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina are Gram-stain-negative, rod-shaped, aerobic and require sodium ions for growth. The major isoprenoid quinone is ubiquinone-8, and iso-branched fatty acids including iso-C17  :  0, summed feature 9 (comprising iso-C17  :  1ω9c and/or 10-methyl C16  :  0) and iso-C15  :  0 are the major cellular fatty acid components [1]. This article describes a haloalkaliphilic strain isolated from a soda lake. The strain is proposed as a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina.,Strain IM 1326T was isolated from a brine sample of a soda lake (Hutong Qagan Lake) from the Ordos, Inner Mongolia Autonomous Region of China, in July 2017. The sampling site was located at the position of 39° 12′ 14″ N, 109° 0′ 8″ E, and an altitude of 1250 m. The total salinity of the lake water was 26%, while the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was 9.95. The brine sample was stored in a sterilized plastic bottle and transported to the laboratory at ambient temperature. The sample was incubated at 37 °C for 2 weeks after being inoculated in LN liquid medium (originally designed for microbe using light by refering neutral oligotrophic medium) composed of (per litre): 15.0 g NaCl, 0.5 g yeast extract (Difco), 6.0 g NaHCO3, 4.0 g Na2CO3, 0.2 g NH4Cl, 2.0 g KCl, 0.25 g fish peptone, 0.38 g sodium formate, 0.25 g sodium acetate, 0.25 g sodium pyruvate, 1.0 g MgSO4·7H2O, 0.08 g CaCl2, 56 mg FeSO4·7H2O, and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was adjusted to 9.5 with 1 M NaOH. The enrichment culture was spread on agar plates of the same medium (15.0 g agar per litre) and incubated at 37 °C for 5 days. Pure cultures were obtained by repeated subculturing of a single colony on the same solid medium. A circular, convex and light brown colony was obtained on LN plates after 5 days of incubation at 37 °C. The strain was routinely cultured at 37 °C in LN medium. Stock culture of the isolate was preserved in LN medium with 20 % glycerol (v/v) at −80 °C.,The phylogenetic position of strain IM 1326T was determined through 16S rRNA gene sequence analysis. The 16S rRNA gene sequence was amplified using oligonucleotide primers complementary to highly conserved regions of bacterial 16S rRNA genes. The forward and reverse primers were 27F (5′-AGAGTTTGATCCTGGCTCAG-3′) and 1492R (5′-GGTTACCTTGTTACGACTT-3′), respectively [13]. The amplified product was cloned in the pMD-18T vector (Takara), and the recombinant plasmids were transformed into \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Escherichia coli\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " JM109 (BioMed) cells. Sanger sequencing was performed by Witon Information Technology Co., Ltd. (Beijing, PR China). Sequences were preliminarily analysed by the blast program (https://blast.ncbi.nlm.nih.gov/Blast.cgi) and the EzBioCloud webserver (www.ezbiocloud.net), and the closely related sequences were selected as references from the list of hits with valid names. The maximum-likelihood, neighbour-joining, minimum-evolution and maximum-parsimony methods were used to reconstruct the phylogenetic trees in mega 6.0 software [14]. The robustness of the phylogenetic tree topology was evaluated by bootstrap analyses based on 1000 replicates [15].,This study determined the nearly full-length 16S rRNA gene sequence (1503 bp) of strain IM 1326T. The similarities between strain IM 1326T and the most closely related \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " A. sanyensis, A. shirensis, A. iranensis, A. haloalkalitolerans, A. celeris, A. minuta and A. taiwanensis (the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of this \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ") were 95.8, 95.7, 95.4, 95.3, 95.0, 94.7 and 93.6%, respectively. In the maximum-likelihood tree, strain IM 1326T fell within a clade comprising \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina. The subclade was composed of A. taiwanensis AIT1T, A. sanyensis GYP-17T, and A. haloalkalitolerans AK5T (Fig. 1). Considering the low bootstrap values (not shown if less than 70%), the topology of the phylogenetic tree was not stable. The topology and phylogenetic relationship were also supported by the neighbour-joining, minimum-evolution and maximum-parsimony phylogenetic trees (Figs S1–S3, available in the online version of this article). In addition, the genera Pseudidiomarina and Idiomarina were closer in most phylogenetic trees (Figs 1, S1 and S2), apart from the maximum-parsimony tree (Fig. S3). Moreover, the bootstrap values were lower than 70 % between the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina and the clade of the genera Pseudidiomarina and Idiomarina. This finding is different from the previous report using FastTree with JTT+CAT parameters [3]. Altogether, strain IM 1326T was likely to represent a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina.,,,The draft genome of strain IM 1326T was sequenced by Biomarker Technologies (Beijing, PR China) for genome-level phylogenetic analysis. Briefly, the genomic DNA was extracted using the cetyltrimethylammonium bromide (CTAB) method, and purity was ascertained using a Qubit 3.0 fluorometer (Invitrogen). After the construction of the DNA library, the genome was sequenced using an Illumina NovaSeq platform. Genome assembly was carried out using SPAdes version 3.13.0. The draft genome sequence has been deposited with accession number VJWL00000000 in the GenBank of the NCBI nucleotide database (www.ncbi.nlm.nih.gov/genome) and annotated using the NCBI Prokaryotic Genome Annotation Pipeline version 4.7 (www.ncbi.nlm.nih.gov/genome/annotation_prok/). DNA G+C content and other detailed information were obtained from the genome sequence. The average nucleotide identity (ANI) percentages with other \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina were calculated using the blast (ANIb) and MUMmer (ANIm) algorithms from the JSpecies Web Server (http://jspecies.ribohost.com/jspeciesws) [16]. The average amino acid identity (AAI) values were calculated using the online tool ANI/AAI-Matrix (http://enve-omics.ce.gatech.edu/g-matrix/), a genome-based distance matrix calculator. In addition, genome-to-genome distance analysis via digital DNA–DNA hybridization (dDDH) was performed with the Genome-to-Genome Distance Calculator (http://ggdc.dsmz.de/distcalc2.php) using the method described by Meier-Kolthoff et al. [17]. Furthermore, a phylogenomic tree was reconstructed based on 120 bacterial conserved single-copy genes obtained by employing the Genome Taxonomy Database [18].,Genome sequencing of strain IM 1326T resulted in raw data of 3.06 Gb with 1189× genome coverage. Sequence reads were de novo assembled into 19 contigs and 12 scaffolds. The genomic size was 2.57 Mb. The genomic DNA G+C content of strain IM 1326T was estimated to be 49.7 mol%, which was similar to that of other \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina (46.3–52.1 mol%) and slightly higher than that of A. taiwanensis AIT1T (48.7 mol%), the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina (Table 1). The detailed genomic information is presented in Table S1. The value of genome relatedness between strain IM 1326T and its closely related \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " ranged from 68.1 to 72.6 % for ANIb, 80.7 to 84.9 % for ANIm, 76 to 78 % for AAI, and 16.2 to 22.7 % for dDDH (Table 2). The percentage similarity values with A. sanyensis GYP-17T, the most closely related \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ", were 72.6 % for ANIb, 83.5 % for ANIm, 78 % for AAI, and 18.4 % for dDDH. The obtained similarity percentage values are lower than the standard thresholds for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " delineation, i.e., 95 % for ANI, 95 % for AAI and 70 % for dDDH.,The maximum-likelihood phylogenomic tree reconstruction based on 120 bacterial conserved single-copy genes revealed that strain IM 1326T was located in the Aliidiomarina clade and separated from other \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " (Fig. 2). The bootstrap value between strain IM 1326T and A. sanyensis GYP-17T was 100%, suggesting it was a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of Aliidiomarina. The neighbour-joining, minimum-evolution and maximum-parsimony phylogenomic trees displayed similar phylogenetic positions (Figs S4–S6). In addition, the phylogenomic trees based on 120 bacterial conserved single-copy proteins agreed with the tree based on the 78 concatenated core protein sequences of the family Idiomarinaceae [3]. Therefore, the results confirmed that IM 1326T represents a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina.,,,The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina, A. taiwanensis CGMCC 1.15255T (=AIT1T, obtained from the China General Microbiological Culture Collection Center), and A. haloalkalitolerans DSM 24939T (=AK5T, obtained from the Leibniz Institute German Collection of Microorganisms and Cell Cultures) were selected as reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " to compare physiological characteristics and fatty acid compositions. A. taiwanensis CGMCC 1.15255T and A. haloalkalitolerans DSM 24939T were cultured using marine broth medium 2216E [peptone (5.0 g l−1), yeast extract (1.0 g l−1), ferric citrate (0.1 g l−1), NaCl (19.45 g l−1), MgCl2 (5.98 g l−1), Na2SO4 (3.24 g l−1), CaCl2 (1.8 g l−1), KCl (0.55 g l−1), Na2CO3 (0.16 g l−1), KBr (0.08 g l−1), SrCl (34.0 mg l−1), boric acid (22.0 mg l−1), sodium silicate (4.0 mg l−1), sodium fluoride (2.4 mg l−1), NaNO3 (1.6 mg l−1), Na2HPO4 (8.0 mg l−1), \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 7.4–7.8] at 30 °C.,The physiological and chemotaxonomic characteristics of strain IM 1326T were examined by using routine cultivation methods on HM medium containing (g l−1): NaCl (30.0), KCl (2.0), MgSO4·7H2O (1.0), NaBr (0.23), CaCl2 (0.27), NaHCO3 (0.06), FeCl3 (trace), tryptone (5.0), yeast extract (10.0) and agar (15.0 for solid medium) [19] at 37 °C and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " adjusted to \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 8.5 before sterilization. The Gram staining analysis was performed according to the method described by Dussault [20]. Morphology and cell size of the cells were observed under a scanning electron microscope (Hitachi H-700A). \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Motility\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was observed using an Olympus BX 51 microscope equipped with phase-contrast optics. The flagellum was observed under a transmission electron microscope. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    temperature range\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " for growth was determined at 1, 4, 15, 20, 25, 30, 34, 37, 40, 42, 45 and 50 °C in HM medium containing 3 % (w/v) NaCl at \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 9.0. The salinity range for growth was observed with 0–15.0 % (w/v) NaCl at intervals of 1.0 % (w/v). After the identification of the optimum salinity, NaCl \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    concentration\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was changed to 4 % (w/v). The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " range for growth was determined from \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 5.0 to 12.0 at intervals of 0.5 \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " units using the following buffers (25 mM \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    concentration\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       "): MES (for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 5.0–6.0), PIPES (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 6.5–7.0), HEPES (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 6.8–8.0), Tricine (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 7.4–8.8), CHES (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 8.6–10.0) and CAPS (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 9.7–12.0). Anaerobic growth was tested in the presence of 5.0 % (w/v) nitrate, 3.0 % (w/v) l-arginine and 5.0 % (v/v) DMSO in stoppered tubes. Tests for catalase and oxidase activities and the hydrolysis of starch, gelatin, casein, and Tweens 20, 40, 60, and 80 were carried out using routine cultivation methods on HM medium with 4 % (w/v) NaCl as described by Harrigan et al. [21, 22]. H2S formation was detected using a filter-paper strip impregnated with lead acetate [23]. Indole \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " from tryptophan, the utilization of sugars and organic acids, and the Voges–Proskauer test were assessed as described by Oren et al. [24]. In brief, the utilization of sole carbon and energy sources was detected using minimal medium (tryptone was omitted and yeast extract was decreased to 1 g l−1) with various supplementary carbon sources and then comparing the cell density. Acid \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " from sugars and alcohols was determined, and the substrates used were d-glucose, d-galactose, sucrose, d-sorbitol, mannose, maltose, starch, d-fructose, l-sorbose, ribose, trehalose, d-xylose and glycerol. Production of acid from the above-mentioned carbohydrates and sugar alcohols was tested on the growth medium supplemented with 10.0 g l−1 of the substrate. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the cultures was measured with a glass electrode at the end of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    incubation period\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " and compared with the initial \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " and the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the control tube without substrate. Reduction of nitrate and nitrite were detected by using sulfanilic acid and α-naphthylamine reagent [25]. All tests were performed in triplicate along with the reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " under identical conditions.,Strain IM 1326T was Gram-stain-negative. The cells were approximately 0.3–0.6 µm wide and 0.8–2.1 µm long (Fig. S7). The isolate can move, but no flagellum was observed under the transmission electron microscope (Fig. S8). Furthermore, the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    motility\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was explored. There were no rafts of cells and isolated cells were observed in the microscope image of the edge of colonies (Fig. S9), indicating that the cells did not move by gliding; therefore, further exploration is necessary. The colonies were light brown when grown on HM medium (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 8.5) for 24 h at 37 °C. Strain IM 1326T was able to grow in presence of 0–13.0 % NaCl \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    concentration\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " (optimum, 4.0–6.0 %), at 4–42 °C (optimum, 37 °C) and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 7.5–11.0 (optimum, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 9.0–10.0). Moreover, it showed more tolerance to alkalinity (up to \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 11.0) than most of the reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ". Strain IM 1326T could hydrolyse gelatin, casein and different kinds of Tweens, e.g. Tween 20, 40 and 60, but could not hydrolyse starch. In addition, strain IM 1326T was positive for nitrate reduction, but negative for nitrite reduction. Cells of strain IM 1326T were oxidase- and catalase-positive. Correspondingly, a respiratory nitrate reductase operon (including four genes FM042_RS01865 to FM042_RS01880), NarK family nitrate/nitrite transporter (FM042_RS01935) and catalase (FM042_RS03320 and FM042_RS09450) were annotated in the genome. However, the draft genome did not contain the gene for amylase and nitrite reduction. Strain IM 1326T could produce H2S from cysteine but did not produce indole from tryptophan. The detailed morphological, physiological and biochemical characteristics of strain IM 1326T are provided in the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    description\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " and Table 1.,Cells grown on HM plates for 2 days at 37 °C were harvested for the cellular fatty acid analysis. The fatty acids were extracted according to the standard midi protocol (Sherlock Microbial Identification System, version 6.0). The cellular fatty acid profiles of strain IM 1326T, A. taiwanensis CGMCC 1.15255T and A. haloalkalitolerans DSM 24939T are shown in Table S2. The predominant fatty acids detected in strain IM 1326T were 10-methyl-C16 : 0/iso-C17 : 1 ω9c (summed feature 9, 22.2 %), iso-C15 : 0 (16.1 %) and iso-C17 : 0 (13.1 %). These are also the dominant fatty acids in the two reference strains. In addition, we observed that, contrary to the original report, the content of C16 : 0 was high (up to 13.2 %) in A. taiwanensis CGMCC 1.15255T [1]. However, the amount of C16 : 0 was lower in IM 1326T and A. haloalkalitolerans DSM24939T (Table S2). These data on the fatty acids composition of two reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " were similar to the previous report [2]. Despite the divergence in results from the different labs, all three \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " contained the major fatty acids of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina [1].,Polar lipids of strain IM 1326T were extracted by a phenol–chloroform system and analysed by using two-dimensional thin-layer chromatography (TLC) as described previously [26]. Merck silica gel 60 F254 aluminum-backed thin-layer plates were used for the TLC analysis. The first and second dimensional TLC solvents were chloroform–methanol–water (65 : 25 : 4, by vol.) and chloroform–methanol–acetate–water (80 : 12 : 15 : 4, by vol.), respectively. The reagent sulfuric acid–ethanol (1 : 1, v/v) was used to detect total polar lipids. According to the TLC chromatogram, the major polar lipids of strain IM 1326T were diphosphatidylglycerol, phosphatidylglycerol, phosphatidylethanolamine and one unidentified aminophospholipid (Fig. S10). The polar lipid composition of two reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " (including the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ") were identified under identical experimental condition (Fig. S10). Most of the Aliidiomarina \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " contained diphosphatidylglycerol, phosphatidylglycerol and phosphatidylethanolamine (Table 1). The above findings are in unison with the polar lipid compositions of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina [1].,Respiratory quinones were extracted from lyophilized cells, and the extracts were purified by TLC and analysed by high-performance liquid chromatography according to the method of Collins [27]. Strain IM 1326T contained ubiquinone-8 as the sole respiratory quinone (Table 1). However, a minor quinone of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina ubiquinone-9 was not detected [1].,Consequently, on the basis of the phylogenetic, physiological and chemotaxonomic data, strain IM 1326T is considered to represent a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina, for which the name Aliidiomarina halalkaliphila sp. nov. is proposed.,Aliidiomarina halalkaliphila (hal.al.ka.li’phi.la. Gr. masc. n. hals halos \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    salt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       "; N.L. n. alkali alkali; Gr. masc. adj. philus friend, loving; N.L. fem. adj. halalkaliphila \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    salt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " and alkali-loving).,Cells are Gram-stain-negative, motile, rod-shaped, approximately 0.3–0.6 µm wide and 0.8–2.1 µm long. Colonies grown for 24 h at 37 °C on HM medium are circular, smooth, convex, light brown and approximately 1.0–3.0 mm in diameter. Oxidase and catalase are positive. Growth occurs at 4–45 °C (optimum, 37 °C), at \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 7.5–11.0 (optimum, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 9.0–10.0) and with 0–13.0 % (w/v) NaCl \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    concentration\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " (optimum 4.0–6.0 %). Anaerobic and microaerobic growth does not occur on HM medium with or without nitrate. Nitrate is reduced, but nitrite is not. H2S is produced from cysteine. Production of indole and utilization of urea does not occur. The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl red\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " reaction is negative, but the Voges–Proskauer test result is positive. Gelatin, casein, DNA and Tweens 20, 40 and 60 are hydrolysed, but Tween 80, aesculin and starch are not hydrolysed. The following substrates are utilized as single carbon and energy sources for growth: l-arginine, l-lactate, l-glutamate, acetate, l-malate, pyruvate, succinate, citrate, l-serine, glycine, valine and histidine. The following substrates are not utilized as sole sources of carbon and energy: d-glucose, d-galactose, sucrose, d-sorbitol, fumarate, l-aspartate, mannose, maltose, starch, l-alanine, l-ornithine, d-fructose, l-sorbose, ribose, trehalose, d-xylose, glycerol and l-lysine. No acids are produced with the following substrates: d-glucose, d-galactose, sucrose, d-sorbitol, mannose, maltose, starch, d-fructose, l-sorbose, ribose, trehalose, d-xylose and glycerol. The respiratory quinone is ubiquinone-8. The major cellular fatty acids of the isolate are 10-methyl-C16 : 0/iso-C17 : 1 ω9c (summed feature 9), iso-C15 : 0 and iso-C17 : 0. The major polar lipids include diphosphatidylglycerol, phosphatidylglycerol, phosphatidylethanolamine and one unidentified aminophospholipid.,The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain, IM 1326T (=CGMCC 1.17056T=JCM 34227T), was isolated from the soda lake Hutong Qagan Lake in Inner Mongolia Autonomous Region, China. The genomic size of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain is 2.57 Mb, and DNA G+C content is 49.7 mol% (determined from the genome sequence).,This study was supported by the National Natural Science Foundation of China (No. 91751201) and the Science and Technology Basic Resources Investigation Project (No. 2017FY100300).,The authors declare that there are no conflicts of interest.,Differentiating characteristics of strain IM 1326T and the ten \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strains of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " AliidiomarinaStrains: 1, IM 1326T (data from this study); 2, A. taiwanensis CGMCC 1.15255T; 3, A. haloalkalitolerans DSM 24939T; 4, A. sanyensis GYP-17T; 5, A. shirensis AIST; 6, A. minuta MLST1T; 7, A. iranensis GBPy7T; 8, A. soli Y4G10-17T; 9, A. maris CF12-14T; 10, A. celeris F3105T; 11, A. sedimenti GBSy1T. +, Positive; −, negative; w, weakly positive; nd, not detected; DPG, diphosphatidylglycerol; PG, phosphatidylglycerol; PE: phosphatidylethanolamine; APL, unidentified aminophospholipid; PL, unidentified phospholipid. DNA G+C content of strain IM 1326T was obtained from the genomic sequence. Data for A. taiwanensis CGMCC 1.15255T, and A. haloalkalitolerans DSM 24939T were taken from Srinivas et al. [2], and some experiments were repeated under identical conditions (marked by *). Data for other \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " were taken from the literature: A. sanyensis GYP-17T [6], A. shirensis AIST [3], A. minuta MLST1T [4], A. iranensis GBPy7T [5], A. soli Y4G10-17T [7], A. maris CF12-14T [8], A. celeris F3105T [9], and A. sedimenti GBSy1T [10].,Genome relatedness based on average nucleotide identity (ANI), average amino acid identity (AAI), and digital DNA–DNA hybridization (dDDH) values among strain IM 1326T and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Aliidiomarina.Strain: 1, IM 1326T; 2, A. taiwanensis CGMCC 1.15255T; 3, A. haloalkalitolerans DSM 24939T; 4, A. sanyensis GYP-17T; 5, A. AIST; 6, A. minuta MLST1T; 7, A. iranensis GBPy7T; 8, A. soli Y4G10-17T; 9, A. maris CF12-14T; 10, A. celeris F3105T; 11, A. sedimenti GBSy1T. ANIb and ANIm values were calculated by blast and MUMmer algorithms, respectively. AAI values were calculated by ANI/AAI-Matrix. dDDH values were obtained by calculating genome-to-genome distance. nan nan ANI, average nucleotide identity; dDDH, digital DNA–DNA hybridization; PHA, poly-β-hydroxyalkanoate.,The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella was proposed by Zheng et al. in 2010, with \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mameliella alba\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " as the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " [1]. In 2021, the ‘Roseobacter clade’ of the family Rhodobacteraceae was validly published as the family Roseobacteraceae [2, 3], therefore the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella was moved to the family Roseobacteraceae of the class Alphaproteobacteria. In 2015, four heterotypic synonyms of \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mameliella alba\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       ", Mameliella phaeodactyli [4], Mameliella atlantica [5], Ponticoccus lacteus [6] and Alkalimicrobium pacificum [7], have been published independently, because the chimaeric 16S rRNA gene sequence of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain JLT354-WT (EU734592.1) had been submitted to the GenBank database [8]. After identification of this mistake, by January 2022, the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella contains only one \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " with a correct validly published name according to List of Prokaryotic names with Standing in Nomenclature (https://lpsn.dsmz.de/genus/mameliella).,During a course of study on the poly-β-hydroxyalkanoate-producing marine bacteria from Daya Bay, Guangdong, PR China, we isolated a novel bacterial strain resembling members of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella, designated DP3N28-2T. Using PCR with specific primers [9], we identified that DP3N28-2T had a poly-β-hydroxyalkanoate (PHA) polymerase (phaC) gene (gel profile data not shown), which indicated the potential for producing PHA. Therefore, in this study, we were aimed to determine the exact taxonomic position of DP3N28-2T, by using a polyphasic approach based on phenotypic, chemotaxonomic and genotypic data, and estimated the PHA-producing \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    ability\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella.,The novel isolate of a member of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella was isolated from a surface sediment sample near off-shore sewage outfall (22°34′19.57″ N, 114°30′29.33″ E) from 5.3 m depth. The sediment sample was incubated in medium containing 5 g (w/w) peptone and 1 g (w/w) yeast extract per litre sea water, at 30 °C for 28 days, and then isolated using the standard dilution plating technique on marine agar (MA; Haibo). After purification by re-streaking, the novel strain was routinely cultivated at 30 °C on MA and stored at −80 °C in sterile 15 % (v/v) glycerol supplemented with 1 % (w/v) saline. The reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain, M. alba CGMCC 1.7209T, were procured from the China General Microbiological Culture Collection Centre. Both strains could form single colonies on MA within 24 h at 30 °C. To standardize the culture conditions in this study, we cultivated M. alba CGMCC 1.7209T using MA or MB.,The 16S rRNA gene of DP3N28-2T was amplified by PCR with the universal primers 27F and 1492R [10], and sequenced by Sangon (Shanghai, PR China). The 16S rRNA gene sequences of DP3N28-2T was analysed by using the blast programme (NCBI) and compared with closely related sequences of reference organisms using the service from EzBioCloud (www.ezbiocloud. net) [11]. Alignments of 16S rRNA gene sequences were performed using the programme clustal_x, version 1.81 [12], and positions with insertions or deletions were excluded during calculations. Phylogenetic trees were reconstructed by the neighbor-joining (NJ) [13], maximum-parsimony (MP) [14] and maximum-likelihood (ML) [15] methods with the mega 7 programme package [16]. The NJ tree and ML tree were reconstructed using evolutionary distances that were calculated with the Kimura two-parameter model with Gamma distribution [17]. The MP tree was obtained using the subtree-pruning–regrafting algorithm [14]. All positions containing gaps and missing data were eliminated. The stability of the clusters was evaluated by bootstrap analysis based on 1000 replicates. A nearly complete 16S rRNA gene sequence of DP3N28-2T obtained by amplification (1303 bp) was included in the 16S rRNA gene sequence assembled from genomic sequences (1437 bp). On the basis of the 16S rRNA gene sequence (from genome sequence), strain DP3N28-2T was closely related to \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Mameliella alba\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " DSM 23384T (98.3%), \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Antarctobacter jejuensis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " 13–2-B6T (97.2%), \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Antarctobacter heliothermus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " El-219T (96.8%), \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Maliponia aquimaris\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " MM-10T (96.7%), \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Ponticoccus litoralis\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " CL-GR66T (96.4%) and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    Aquicoccus porphyridii\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">SPECIES</span>\n",
       "</mark>\n",
       " L1 8-17T (96.1%). In the NJ, MP and ML phylogenetic trees (Fig. 1), although the distance between DP3N28-2T and M. alba was the shortest, they did not cluster together. Therefore, a genome-based phylogenetic tree was reconstructed using the Type (Strain) Genome Server provided by DSMZ [18] to further verify the phylogenetic position. This tree indicated that DP3N28-2T formed a clade with the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain of M. alba (Fig. 2), which indicated that DP3N28-2T could represent a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella.,,,,,The draft genome of DP3N28-2T was sequenced using a HiSeq X-ten platform (Illumina), assembled at Guangdong Magigene Biotechnology. (Guangdong, PR China; http://www.magigene.com/) using the SPAdes software (v3.13.0) [19] and annotated using the NCBI Prokaryotic Genome Annotation Pipeline [20]. The assembled genome of DP3N28-2T (JAHUZG000000000) had a total length of 5 120 735 bp, coverage of 600×, contig N50 value of 1 507 360 bp, 4966 predicted gene sequences and 54 RNAs. The final assembly contained 21 contigs and a had a DNA G+C content of 63.0 mol%.,Average nucleotide identity (ANI) using the OrthoANI algorithm [21] and digital DNA–DNA hybridization (dDDH) values [22] was obtained to determine the genetic relatedness between DP3N28-2T and the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " M. alba CGMCC 1.7290T (FMZI01000000). ANI and dDDH (formula 2) were 81.1 and 23.4 %, respectively. They were both lower than the threshold value for \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " demarcation (ANI, 95–96 %; dDDH, 70 %). Furthermore, the difference in DNA G+C content between DP3N28-2T and the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was 2.2 mol%, which indicated that they represented two different \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " [23]. Therefore, on the basis of the results of the phylogenetic analysis, the 16S rRNA gene sequence similarity values and the genomic sequence similarity values, DP3N28-2T was considered to represent a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella.,All morphological and physiological features of DP3N28-2T were examined after incubation in MB or on MA at 30 °C for 24 h to a maximum of 1 week unless otherwise specified. Cell morphology, Gram-staining and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    motility\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " were determined by microscopy (CX43, Olympus) and transmission electron microscopy (Tecnai G2 F30, FEI). Cells of DP3N28-2T were Gram-stain-negative non-flagellated and rod shaped. The cell length was 1.3–4.7 µm and the width was 0.5–0.9 µm (Fig. S1, available in the online version of this article). The shape varied due to the distribution of the PHA granules. Growth of DP3N28-2T and M. alba CGMCC 1.7290T was tested at 4, 8, 15, 20, 25, 30, 35, 37, 40, 45 and 50 °C, NaCl concentrations in the range of 0–10 % (w/v; at increments of 1 %) and \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " values in the range of 4.0–12.0 (at intervals of 1.0 \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " unit) adjusted using HCl or NaOH. The growth was measured using OD600 after 24, 48 and 96 h. Growth occurred at temperatures from 15 to 45 °C (optimally at 37–40 °C), \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " from 5.0 to 8.0 (optimally at \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 6.0) and NaCl concentrations from 0–10 % (optimally at 4 %). No growth observed after incubation in an anaerobic chamber on MA for 2 weeks. Oxidase \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    activity\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was determined using a bioMérieux oxidase reagent kit according to the manufacturer’s instructions. Catalase \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    activity\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " was tested by bubble \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " using 3 % (v/v) H2O2 solution. Utilization of citrate, pyruvate and malonate as sole carbon sources was determined. DP3N28-2T was cultivated in the minimal \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    salt\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " medium containing, per litre of distilled water: 2.28 g K2HPO4·3H2O, 1.32 g (NH4)2SO4, 0.47 g NaHPO4, supplemented with 1 % carbon source. Hydrolysis of agar and starch, H2S \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ", reduction of nitrate, Voges–Proskauer test, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl red\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " test and indole \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " were investigated according to the methods described by Dong and Cai [24]. The phenotypic characterization was carried out using API 20NE, API ZYM (bioMérieux) and GEN III MicroPlate (Biolog) on DP3N28-2T and the reference \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain independently twice without replicates at 30 °C. The polyhydroxyalkanoates \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of DP3N28-2T was examined as described by Juengert et al. [25] using gas chromatography (GC-2014, Shimadzu) with GC column DB-WAX (G6501-CTC, Agilent Technologies).,DP3N28-2T exhibited phenotypic similarities to M. alba CGMCC 1.7290T, including being Gram-stain-negative, rod shaped, aerobic, having positive reactions for catalase and oxidase and identical reactions in API ZYM tests, except for α-galactosidase. Both strains could accumulate poly-β-hydroxybutyrate. The GC result indicated that DP3N28-2T and M. alba CGMCC 1.7290T accumulated PHA up to 12.3±2.9 % and 19.2±5.8 % (per dry cell weight, ±standard deviation), respectively. In the genomic data, the PHA synthase gene (phaC), PHA depolymerase gene (phaZ), regulator gene (phaR) and PHA granule-associated phasin gene (phaP) were annotated (accession numbers are listed in Table S1).,On the basis of the data obtained during this study, we could distinguish DP3N28-2T from M. alba in various ways (Table 1). However, the reported phenotypic features of M. alba were highly variable among the five representatives [8], for example the optimal NaCl (w/v, %) of M. alba JLT354-WT was 1–3 [1] while that of A. pacificum F15T was 3.5 [7], from which we could draw different conclusion about the similarity between strain DP3N28-2T and M. alba. Therefore, we identified three features including being slightly curved, negative for nitrate reduction and preferring acidic growth conditions (\n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 6.0) as the critical phenotypic differences between DP3N28-2T and M. alba. The detailed physiological and biochemical characteristics of DP3N28-2T are given in Tables 1 and 2, Table S2 and the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " descriptions. According to the Biolog GEN III MicroPlate test, DP3N28-2T was susceptible to troleandomycin, nalidixic acid, aztreonam and niaproof 4, and resistant to lincomycin, vancomycin, fusidic acid, rifamycin SV, guanidine HCl and minocycline. Cultures of DP3N28-2T and M. alba CGMCC 1.7290T were grown in MB at 30 °C for 48 h, and then freeze-dried for polar lipid, respiratory quinone and cellular fatty acid composition analysis. The cellular fatty acid composition was measured by the Marine Culture Collection of China (MCCC; Xiamen, Fujian, China) using the Microbial ID system [26]. Respiratory quinones of DP3N28-2T was determined by the MCCC using ultra-performance liquid chromatography–tandem mass spectrometry methods [27]. The cellular polar lipids were extracted and determined by two-dimensional thin-layer chromatography using silica gel plates (60 F254, Merck) [28, 29]. Total lipid material was detected using molybdatophosphoric acid, and specific functional groups were detected using ninhydrin and phosphomolybdenum blue reagents [29, 30]. The respiratory quinone of both strains was Q-10. As shown in Table 2, the two strains showed similar cellular fatty acid profiles, with summed feature 8, C16 : 0 and C18 : 0 as the top three components (&amp;gt;5 %), however, in M. alba CGMCC 1.7290T, the proportion of C18 : 0 was higher than that of C16 : 0, which were consistently shown in all the five representatives of M. alba [8]. As shown in Fig. S2, the polar lipids of DP3N28-2T included phosphatidylethanolamine, phosphatidylmonomethylethanolamine, phosphatidylglycerol, one aminophospholipid, one phospholipid and three unidentified lipids. The polar lipid composition was variable among the M. alba strains [8], but none of them contained phosphatidylmonomethylethanolamine, which could be used to distinguish DP3N28-2T from M. alba.,On the basis of these phenotypic, chemotaxonomic, phylogenetic and genetic data, we propose to classify strain DP3N28-2T as representing a novel \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella, for which the name Mameliella sediminis sp. nov. is proposed.,Mameliella sediminis (se.di’mi.nis L. gen. neut. n. sediminis, of sediment).,Cells are Gram-stain-negative, aerobic, and non-motile rods (0.5–0.9 µm wide and 1.3–4.7 µm long). Small colonies (1–2 mm) are produced on MA after incubation at 30 °C for 24 h. Colonies are smooth, convex and beige. Growth occurs at 15–45 °C, at \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 5.0–8.0 and in the presence of 0–10 % NaCl; optimal growth occurs at 37–40 °C, at \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    pH\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " 6.0 and in the presence of 4.0 % NaCl. Positive for catalase and oxidase. Negative for nitrate reduction, hydrolysis of gelatin and starch, \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    methyl red\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " test, Voges–Proskauer reaction, and H2S \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ". Pyruvate are used as sole carbon source, but not citrate. In the API ZYM tests, positive for alkaline phosphatase, esterase (C4), esterase lipase (C8), lipase (C14), leucine arylamidase, valine arylamidase, cystine arylamidase, acid phosphatase, naphthol-AS-BI-phosphohydrolase and α-galactosidase; negative for trypsin, α-chymotrypsin, β-galactosidase, β-glucuronidase, α-glucosidase, β-glucosidase, N-acetyl-β-glucosaminidase, α-mannosidase and α-fucosidase. In the API 20NE tests, positive for d-glucose fermentation, urease, β-glucosidase (aesculin hydrolysis) and β-galactosidase; weakly positive for gelatin hydrolysis, utilization of d-glucose, l-arabinose, and d-mannose; negative for reduction of nitrate to nitrite, denitrification, indole \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    production\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ", arginine dihydrolase, utilization of d-mannitol, N-acetyl-glucosamine, maltose, potassium gluconate, capric acid, malic acid, adipic acid, phenylacetic acid and trisodium citrate. According to Biolog GEN III MicroPlate assays, α-hydroxybutyric acid, l-arginine, l-lactic acid, β-hydroxy-d,l-butyric acid, d-fucose, acetoacetic acid and trehalose are oxidized. The polar lipids are phosphatidylethanolamine, phosphatidylmonomethylethanolamine, phosphatidylglycerol, one aminophosphlipid, one phospholipid and three unidentified lipids. The main fatty acids are summed feature 8 (C18 : 1ω6c and/or C18 : 1ω7c) and C16 : 0. The respiratory quinone is Q-10.,The \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain, DP3N28-2T (=MCCC 1K06218T=KCTC 82804T) was isolated from the sediment collected from Daya Bay, Guangdong, PR China. The DNA G+C content of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " strain is 63.0 mol % (data from genome sequence). The GenBank accession number for the 16S rRNA gene sequence and draft genome of strain DP3N28-2T are MZ277401 and JAHUZG000000000 respectively.,This work was supported by the Key Program of Marine Economy Development (Six Marine Industries) Special Foundation of Department of Natural Resources of Guangdong Province (GDNRC[2020]034) and China Postdoctoral Science Foundation (2019M662806).,W.S.Z., S.Q.Z. and Y.H. designed the study; W.S.Z., S.Q.Z. and M.J.Z. performed the experiments; W.S.Z. and S.Q.Z. analysed the data; Y.H. and W.S.Z. supported the research; W.S.Z., S.Q.Z. and Y.H. wrote the paper.,The authors declare that there are no conflicts of interest.,Characteristics of DP3N28-2T and the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    species\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " of the \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    genus\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       " Mameliella. Strain: 1, Mameliella sp. DP3N28−2T; 2, M. alba CGMCC 1.7290T. All Data were obtained from this study unless otherwise indicated. +, Positive or grow; −, negative or not grow; W, weakly positive or grow. The detailed data for API ZYM, API 20NE and Biolog GEN III are given in Table S2.,Cellular fatty acid contents (percentages) of DP3N28-2T and Mameliella albaStrains: 1, DP3N28-2T; 2, M. alba CGMCC 1.7290T. All data were obtained during this study. Major fatty acids (&amp;gt;10%) are highlighted in bold \n",
       "<mark class=\"entity\" style=\"background: #ddd; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;\">\n",
       "    type\n",
       "    <span style=\"font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem\">KEYWORD</span>\n",
       "</mark>\n",
       ". Only fatty acids exceeding 1.0 % of the total cellular fatty acids of at least one of the strains are shown. tr, Trace (&amp;lt;1.0%); nd, not detected. nan nan</div></span>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "spacy.displacy.render(doc, style='ent',jupyter=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b499c71",
   "metadata": {},
   "source": [
    "# Analysing the articles"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "41b70a7f",
   "metadata": {},
   "outputs": [],
   "source": [
    "pd.options.display.max_rows = 4000\n",
    "#pd.options.display.max_seq_items = 2000"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "a32ceda9",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv(\"fulltext_articles.csv\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "37a734b4",
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "#converts df column to list\n",
    "articles = df[\"fulltext\"].head(5).tolist()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "e9927725",
   "metadata": {},
   "outputs": [],
   "source": [
    "# convertrs list to string\n",
    "articles = ' '.join([str(elem) for elem in articles])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "9d6089bb",
   "metadata": {},
   "outputs": [],
   "source": [
    "# erase linebreaks for better nlp results\n",
    "articles = articles.replace(\"\\n\\n\", \" \").replace(\"\\n\", \" \").strip()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d791b7be",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Approach No. 2: maybe faster and easier?\n",
    "df = pd.read_csv(\"fulltext_articles.csv\")\n",
    "list_ft = df.loc[:5, \"fulltext\"].tolist()\n",
    "string_ft = ' '.join(map(str, list_ft))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "805c53ed",
   "metadata": {},
   "outputs": [],
   "source": [
    "nlp = spacy.load(\"en_core_web_trf\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "4e83d361",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "D:\\TH_Koeln\\WS21-22\\Bachelorarbeit\\python_scripts\\venv_spacy\\lib\\site-packages\\torch\\autocast_mode.py:162: UserWarning: User provided device_type of 'cuda', but CUDA is not available. Disabling\n",
      "  warnings.warn('User provided device_type of \\'cuda\\', but CUDA is not available. Disabling')\n"
     ]
    }
   ],
   "source": [
    "doc = nlp(articles)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4590c374",
   "metadata": {},
   "source": [
    "## Lemmatizing of doc object"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "f9480eca",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ANI , average nucleotide identity ; ANIb , average nucleotide identity calculate use blast ; ANIm , average nucleotide identity calculate use MUMmer ; dddh , digital dna – dna hybridization ; dddh , digital dna – dna hybridization . ,the genus Aliidiomarina belong to the family Idiomarinaceae , order Alteromonadales , class Gammaproteobacteria [ 1 ] . this genus have two sister genus name Idiomarina and Pseudidiomarina in the List of prokaryotic name with Standing in Nomenclature . despite the reclassification of the genus Pseudidiomarina into the genus Idiomarina , the specie of both genus could not be distinguish from each other use phenotypic or chemotaxonomic characteristic [ 2 ] , and recently the genus Pseudidiomarina be reinstate follow the result of genome - base analysis [ 3 ] . the genus Aliidiomarina be close to the genus Pseudidiomarina in the phylogenetic tree base on the 16s rrna gene . however , it be distant from both sister genus in the phylogenomic tree base on the 78 core gene [ 3 ] . at the time of write , a total of ten specie of this genus be validly publish , include Aliidiomarina taiwanensi [ 1 ] , Aliidiomarina haloalkalitoleran [ 4 ] , Aliidiomarina shirensi [ 5 ] , Aliidiomarina minuta [ 6 ] , Aliidiomarina iranensi [ 7 ] , Aliidiomarina sanyensi [ 8 ] , Aliidiomarina solo [ 9 ] , aliidiomarina mari [ 10 ] , aliidiomarina celeri [ 11 ] and aliidiomarina sedimenti [ 12 ] . these specie be isolate from a wide variety of habitat , include seawater [ 1 , 4 ] , saline lake [ 5 , 6 ] , coastal - marine wetland [ 7 ] , Spirulina platensis cultivation pool [ 8 ] , soil [ 9 ] and sediment [ 10–12 ] . member of the genus Aliidiomarina be gram - stain - negative , rod - shaped , aerobic and require sodium ion for growth . the major isoprenoid quinone be ubiquinone-8 , and iso - branched fatty acid include iso - c17    :    0 , sum feature 9 ( comprise iso - c17    :    1ω9c and/or 10 - methyl c16    :    0 ) and iso - c15    :    0 be the major cellular fatty acid component [ 1 ] . this article describe a haloalkaliphilic strain isolate from a soda lake . the strain be propose as a novel species of the genus Aliidiomarina . ,strain IM 1326 T be isolate from a brine sample of a soda lake ( Hutong Qagan Lake ) from the Ordos , Inner Mongolia Autonomous Region of China , in July 2017 . the sampling site be locate at the position of 39 ° 12′ 14″ n , 109 ° 0′ 8″ e , and an altitude of 1250   m. the total salinity of the lake water be 26 % , while the ph be 9.95 . the brine sample be store in a sterilize plastic bottle and transport to the laboratory at ambient temperature . the sample be incubate at 37   ° c for 2   week after be inoculate in LN liquid medium ( originally design for microbe use light by refer neutral oligotrophic medium ) compose of ( per litre ): 15.0   g NaCl , 0.5   g yeast extract ( Difco ) , 6.0   g NaHCO3 , 4.0   g na2co3 , 0.2   g nh4cl , 2.0   g kcl , 0.25   g fish peptone , 0.38   g sodium formate , 0.25   g sodium acetate , 0.25   g sodium pyruvate , 1.0   g mgso4·7h2o , 0.08   g cacl2 , 56   mg feso4·7h2o , and ph be adjust to 9.5 with 1 m NaOH . the enrichment culture be spread on agar plate of the same medium ( 15.0   g agar per litre ) and incubate at 37   ° c for 5   day . pure culture be obtain by repeat subculturing of a single colony on the same solid medium . a circular , convex and light brown colony be obtain on LN plate after 5   day of incubation at 37   ° c . the strain be routinely culture at 37   ° c in LN medium . stock culture of the isolate be preserve in LN medium with 20   % glycerol ( v / v ) at −80   ° c.,the phylogenetic position of strain IM 1326 T be determine through 16s rrna gene sequence analysis . the 16s rrna gene sequence be amplify use oligonucleotide primer complementary to highly conserved region of bacterial 16s rrna gene . the forward and reverse primer be 27f ( 5′-agagtttgatcctggctcag-3′ ) and 1492r ( 5′-ggttaccttgttacgactt-3′ ) , respectively [ 13 ] . the amplify product be clone in the pmd-18 t vector ( Takara ) , and the recombinant plasmid be transform into Escherichia coli JM109 ( BioMed ) cell . sanger sequencing be perform by Witon Information Technology Co. , Ltd. ( Beijing , PR China ) . sequence be preliminarily analyse by the blast program ( https://blast.ncbi.nlm.nih.gov/blast.cgi ) and the EzBioCloud webserver ( www.ezbiocloud.net ) , and the closely related sequence be select as reference from the list of hit with valid name . the maximum - likelihood , neighbour - joining , minimum - evolution and maximum - parsimony method be use to reconstruct the phylogenetic tree in mega 6.0 software [ 14 ] . the robustness of the phylogenetic tree topology be evaluate by bootstrap analysis base on 1000 replicate [ 15].,this study determine the nearly full - length 16s rrna gene sequence ( 1503   bp ) of strain IM 1326t. the similarity between strain IM 1326 T and the most closely related specie a. sanyensis , a. shirensi , a. iranensi , a. haloalkalitoleran , a. celeri , a. minuta and a. taiwanensi ( the type specie of this genus ) be 95.8 , 95.7 , 95.4 , 95.3 , 95.0 , 94.7 and 93.6 % , respectively . in the maximum - likelihood tree , strain IM 1326 T fall within a clade comprise specie of the genus Aliidiomarina . the subclade be compose of a. taiwanensis AIT1 T , a. sanyensis GYP-17 T , and a. haloalkalitoleran AK5 T ( fig . 1 ) . consider the low bootstrap value ( not show if less than 70 % ) , the topology of the phylogenetic tree be not stable . the topology and phylogenetic relationship be also support by the neighbour - joining , minimum - evolution and maximum - parsimony phylogenetic tree ( fig s1 – s3 , available in the online version of this article ) . in addition , the genus Pseudidiomarina and Idiomarina be close in most phylogenetic tree ( Figs 1 , s1 and s2 ) , apart from the maximum - parsimony tree ( fig . s3 ) . moreover , the bootstrap value be low than 70   % between the genus Aliidiomarina and the clade of the genus Pseudidiomarina and Idiomarina . this finding be different from the previous report use FastTree with jtt+cat parameter [ 3 ] . altogether , strain IM 1326 T be likely to represent a novel species of the genus Aliidiomarina . ,,,The draft genome of strain IM 1326 T be sequence by Biomarker Technologies ( Beijing , PR China ) for genome - level phylogenetic analysis . briefly , the genomic dna be extract use the cetyltrimethylammonium bromide ( CTAB ) method , and purity be ascertain use a Qubit 3.0 fluorometer ( Invitrogen ) . after the construction of the dna library , the genome be sequence use an Illumina NovaSeq platform . genome assembly be carry out use SPAdes version 3.13.0 . the draft genome sequence have be deposit with accession number VJWL00000000 in the GenBank of the NCBI nucleotide database ( www.ncbi.nlm.nih.gov/genome ) and annotate use the NCBI Prokaryotic Genome Annotation pipeline version 4.7 ( www.ncbi.nlm.nih.gov/genome/annotation_prok/ ) . dna g+c content and other detailed information be obtain from the genome sequence . the average nucleotide identity ( ANI ) percentage with other specie of the genus Aliidiomarina be calculate use the blast ( ANIb ) and MUMmer ( ANIm ) algorithm from the JSpecies Web Server ( http://jspecies.ribohost.com/jspeciesws ) [ 16 ] . the average amino acid identity ( AAI ) value be calculate use the online tool ANI / AAI - Matrix ( http://enve-omics.ce.gatech.edu/g-matrix/ ) , a genome - base distance matrix calculator . in addition , genome - to - genome distance analysis via digital dna – dna hybridization ( dddh ) be perform with the genome - to - genome distance Calculator ( http://ggdc.dsmz.de/distcalc2.php ) use the method describe by Meier - Kolthoff et al . [ 17 ] . furthermore , a phylogenomic tree be reconstruct base on 120 bacterial conserve single - copy gene obtain by employ the Genome Taxonomy Database [ 18].,genome sequencing of strain IM 1326 T result in raw datum of 3.06   gb with 1189× genome coverage . sequence read be de novo assemble into 19 contig and 12 scaffold . the genomic size be 2.57   mb . the genomic dna g+c content of strain IM 1326 T be estimate to be 49.7 mol% , which be similar to that of other specie of the genus Aliidiomarina ( 46.3–52.1   mol% ) and slightly high than that of A. taiwanensis AIT1 T ( 48.7 mol% ) , the type species of the genus Aliidiomarina ( Table 1 ) . the detailed genomic information be present in Table s1 . the value of genome relatedness between strain IM 1326 T and its closely relate specie range from 68.1 to 72.6   % for ANIb , 80.7 to 84.9   % for ANIm , 76 to 78   % for AAI , and 16.2 to 22.7   % for dddh ( Table 2 ) . the percentage similarity value with A. sanyensis GYP-17 T , the most closely related species , be 72.6   % for ANIb , 83.5   % for ANIm , 78   % for AAI , and 18.4   % for dddh . the obtain similarity percentage value be low than the standard threshold for specie delineation , i.e. , 95   % for ANI , 95   % for AAI and 70   % for dddh.,the maximum - likelihood phylogenomic tree reconstruction base on 120 bacterial conserve single - copy gene reveal that strain IM 1326 T be locate in the Aliidiomarina clade and separate from other specie ( fig . 2 ) . the bootstrap value between strain IM 1326 T and A. sanyensis GYP-17 T be 100 % , suggest it be a novel species of Aliidiomarina . the neighbour - join , minimum - evolution and maximum - parsimony phylogenomic tree display similar phylogenetic position ( fig s4 – s6 ) . in addition , the phylogenomic tree base on 120 bacterial conserve single - copy protein agree with the tree base on the 78 concatenate core protein sequence of the family Idiomarinaceae [ 3 ] . therefore , the result confirm that IM 1326 t represent a novel species in the genus Aliidiomarina . ,,,The type specie of the genus Aliidiomarina , a. taiwanensis CGMCC 1.15255 T (= AIT1 T , obtain from the China General Microbiological Culture Collection Center ) , and a. haloalkalitoleran DSM 24939 T (= AK5 T , obtain from the Leibniz Institute German Collection of Microorganisms and Cell Cultures ) be select as reference specie to compare physiological characteristic and fatty acid composition . A. taiwanensi CGMCC 1.15255 T and a. haloalkalitoleran DSM 24939 T be culture use marine broth medium 2216e [ peptone ( 5.0   g l−1 ) , yeast extract ( 1.0   g l−1 ) , ferric citrate ( 0.1   g l−1 ) , NaCl ( 19.45   g l−1 ) , MgCl2 ( 5.98   g l−1 ) , Na2SO4 ( 3.24   g l−1 ) , CaCl2 ( 1.8   g l−1 ) , KCl ( 0.55   g l−1 ) , Na2CO3 ( 0.16   g l−1 ) , KBr ( 0.08   g l−1 ) , SrCl ( 34.0   mg l−1 ) , boric acid ( 22.0   mg l−1 ) , sodium silicate ( 4.0   mg l−1 ) , sodium fluoride ( 2.4   mg l−1 ) , NaNO3 ( 1.6   mg l−1 ) , Na2HPO4 ( 8.0   mg l−1 ) , ph 7.4–7.8 ] at 30   ° c.,the physiological and chemotaxonomic characteristic of strain IM 1326 T be examine by use routine cultivation method on hm medium contain ( g l−1 ): NaCl ( 30.0 ) , KCl ( 2.0 ) , mgso4·7h2o ( 1.0 ) , NaBr ( 0.23 ) , CaCl2 ( 0.27 ) , NaHCO3 ( 0.06 ) , FeCl3 ( trace ) , tryptone ( 5.0 ) , yeast extract ( 10.0 ) and agar ( 15.0 for solid medium ) [ 19 ] at 37   ° c and ph adjust to ph 8.5 before sterilization . the gram staining analysis be perform accord to the method describe by Dussault [ 20 ] . morphology and cell size of the cell be observe under a scan electron microscope ( Hitachi H-700A ) . motility be observe use an Olympus BX 51 microscope equip with phase - contrast optic . the flagellum be observe under a transmission electron microscope . the temperature range for growth be determine at 1 , 4 , 15 , 20 , 25 , 30 , 34 , 37 , 40 , 42 , 45 and 50   ° c in HM medium contain 3   % ( w / v ) NaCl at ph 9.0 . the salinity range for growth be observe with 0–15.0   % ( w / v ) nacl at interval of 1.0   % ( w / v ) . after the identification of the optimum salinity , NaCl concentration be change to 4   % ( w / v ) . the ph range for growth be determine from ph 5.0 to 12.0 at interval of 0.5 ph unit use the following buffer ( 25   mm concentration ): MES ( for ph 5.0–6.0 ) , PIPES ( ph 6.5–7.0 ) , HEPES ( ph 6.8–8.0 ) , Tricine ( ph 7.4–8.8 ) , CHES ( ph 8.6–10.0 ) and CAPS ( ph 9.7–12.0 ) . anaerobic growth be test in the presence of 5.0   % ( w / v ) nitrate , 3.0   % ( w / v ) l - arginine and 5.0   % ( v / v ) dmso in stopper tube . test for catalase and oxidase activity and the hydrolysis of starch , gelatin , casein , and Tweens 20 , 40 , 60 , and 80 be carry out use routine cultivation method on HM medium with 4   % ( w / v ) nacl as describe by Harrigan et al . [ 21 , 22 ] . h2s formation be detect use a filter - paper strip impregnate with lead acetate [ 23 ] . indole production from tryptophan , the utilization of sugar and organic acid , and the Voges – Proskauer test be assess as describe by Oren et al . [ 24 ] . in brief , the utilization of sole carbon and energy source be detect use minimal medium ( tryptone be omit and yeast extract be decrease to 1   g l−1 ) with various supplementary carbon source and then compare the cell density . acid production from sugar and alcohol be determine , and the substrate use be d - glucose , d - galactose , sucrose , d - sorbitol , mannose , maltose , starch , d - fructose , l - sorbose , ribose , trehalose , d - xylose and glycerol . production of acid from the above - mention carbohydrate and sugar alcohol be test on the growth medium supplement with 10.0   g l−1 of the substrate . the ph of the culture be measure with a glass electrode at the end of the incubation period and compare with the initial ph and the ph of the control tube without substrate . reduction of nitrate and nitrite be detect by use sulfanilic acid and α - naphthylamine reagent [ 25 ] . all test be perform in triplicate along with the reference species under identical condition . ,strain IM 1326 T be gram - stain - negative . the cell be approximately 0.3–0.6   µm wide and 0.8–2.1   µm long ( fig . s7 ) . the isolate can move , but no flagellum be observe under the transmission electron microscope ( fig . s8 ) . furthermore , the type of motility be explore . there be no raft of cell and isolate cell be observe in the microscope image of the edge of colony ( fig . s9 ) , indicate that the cell do not move by glide ; therefore , further exploration be necessary . the colony be light brown when grow on hm medium ( ph 8.5 ) for 24   h at 37   ° c . strain IM 1326 T be able to grow in presence of 0–13.0   %   nacl concentration ( optimum , 4.0–6.0   % ) , at 4–42   ° c ( optimum , 37   ° c ) and ph 7.5–11.0 ( optimum , ph 9.0–10.0 ) . moreover , it show more tolerance to alkalinity ( up to ph 11.0 ) than most of the reference specie . strain IM 1326 T could hydrolyse gelatin , casein and different kind of Tweens , e.g. Tween 20 , 40 and 60 , but could not hydrolyse starch . in addition , strain IM 1326 T be positive for nitrate reduction , but negative for nitrite reduction . cell of strain IM 1326 T be oxidase- and catalase - positive . correspondingly , a respiratory nitrate reductase operon ( include four gene fm042_rs01865 to fm042_rs01880 ) , NarK family nitrate / nitrite transporter ( fm042_rs01935 ) and catalase ( fm042_rs03320 and fm042_rs09450 ) be annotate in the genome . however , the draft genome do not contain the gene for amylase and nitrite reduction . strain IM 1326 T could produce h2s from cysteine but do not produce indole from tryptophan . the detailed morphological , physiological and biochemical characteristic of strain IM 1326 T be provide in the species description and Table 1.,cell grow on hm plate for 2   day at 37   ° c be harvest for the cellular fatty acid analysis . the fatty acid be extract accord to the standard midi protocol ( Sherlock Microbial Identification System , version 6.0 ) . the cellular fatty acid profile of strain IM 1326 T , a. taiwanensis CGMCC 1.15255 t and a. haloalkalitoleran DSM 24939 T be show in Table s2 . the predominant fatty acid detect in strain IM 1326 T be 10 - methyl - c16   :   0 / iso - c17   :   1   ω9c ( sum feature 9 , 22.2   % ) , iso - c15   :   0 ( 16.1   % ) and iso - c17   :   0 ( 13.1   % ) . these be also the dominant fatty acid in the two reference strain . in addition , we observe that , contrary to the original report , the content of c16   :   0 be high ( up to 13.2   % ) in a. taiwanensis CGMCC 1.15255 t [ 1 ] . however , the amount of c16   :   0 be low in IM 1326 T and a. haloalkalitoleran dsm24939 T ( table s2 ) . these datum on the fatty acid composition of two reference specie be similar to the previous report [ 2 ] . despite the divergence in result from the different lab , all three specie contain the major fatty acid of the genus Aliidiomarina [ 1].,polar lipid of strain IM 1326 T be extract by a phenol – chloroform system and analyse by use two - dimensional thin - layer chromatography ( TLC ) as describe previously [ 26 ] . Merck silica gel 60   f254 aluminum - back thin - layer plate be use for the TLC analysis . the first and second dimensional TLC solvent be chloroform – methanol – water ( 65   :   25   :   4 , by vol . ) and chloroform – methanol – acetate – water ( 80   :   12   :   15   :   4 , by vol . ) , respectively . the reagent sulfuric acid – ethanol ( 1   :   1 , v / v ) be use to detect total polar lipid . accord to the TLC chromatogram , the major polar lipid of strain IM 1326 T be diphosphatidylglycerol , phosphatidylglycerol , phosphatidylethanolamine and one unidentified aminophospholipid ( fig . s10 ) . the polar lipid composition of two reference specie ( include the type specie ) be identify under identical experimental condition ( fig . s10 ) . Most of the Aliidiomarina specie contain diphosphatidylglycerol , phosphatidylglycerol and phosphatidylethanolamine ( Table 1 ) . the above finding be in unison with the polar lipid composition of the genus Aliidiomarina [ 1].,respiratory quinone be extract from lyophilized cell , and the extract be purify by TLC and analyse by high - performance liquid chromatography accord to the method of Collins [ 27 ] . strain IM 1326 T contain ubiquinone-8 as the sole respiratory quinone ( Table 1 ) . however , a minor quinone of the genus aliidiomarina ubiquinone-9 be not detect [ 1].,consequently , on the basis of the phylogenetic , physiological and chemotaxonomic datum , strain IM 1326 T be consider to represent a novel species of the genus Aliidiomarina , for which the name Aliidiomarina halalkaliphila sp . nov . be propose . ,Aliidiomarina halalkaliphila ( hal.al.ka.li’phi.la . Gr . masc . n. hal halo salt ; N.L. n. alkali alkali ; Gr . masc . adj . philus friend , love ; N.L. fem . adj . halalkaliphila salt and alkali - loving).,Cells be gram - stain - negative , motile , rod - shaped , approximately 0.3–0.6   µm wide and 0.8–2.1   µm long . colony grow for 24   h at 37   ° c on HM medium be circular , smooth , convex , light brown and approximately 1.0–3.0   mm in diameter . oxidase and catalase be positive . growth occur at 4–45   ° c ( optimum , 37   ° c ) , at ph 7.5–11.0 ( optimum , ph 9.0–10.0 ) and with 0–13.0   % ( w / v ) nacl concentration ( optimum 4.0–6.0   % ) . anaerobic and microaerobic growth do not occur on HM medium with or without nitrate . nitrate be reduce , but nitrite be not . h2s be produce from cysteine . production of indole and utilization of urea do not occur . the methyl red reaction be negative , but the Voges – Proskauer test result be positive . gelatin , casein , dna and Tweens 20 , 40 and 60 be hydrolyse , but Tween 80 , aesculin and starch be not hydrolyse . the following substrate be utilize as single carbon and energy source for growth : l - arginine , l - lactate , l - glutamate , acetate , l - malate , pyruvate , succinate , citrate , l - serine , glycine , valine and histidine . the following substrate be not utilize as sole source of carbon and energy : d - glucose , d - galactose , sucrose , d - sorbitol , fumarate , l - aspartate , mannose , maltose , starch , l - alanine , l - ornithine , d - fructose , l - sorbose , ribose , trehalose , d - xylose , glycerol and l - lysine . no acid be produce with the following substrate : d - glucose , d - galactose , sucrose , d - sorbitol , mannose , maltose , starch , d - fructose , l - sorbose , ribose , trehalose , d - xylose and glycerol . the respiratory quinone be ubiquinone-8 . the major cellular fatty acid of the isolate be 10 - methyl - c16   :   0 / iso - c17   :   1   ω9c ( sum feature 9 ) , iso - c15   :   0 and iso - c17   :   0 . the major polar lipid include diphosphatidylglycerol , phosphatidylglycerol , phosphatidylethanolamine and one unidentified aminophospholipid . ,the type strain , IM 1326 T (= CGMCC 1.17056t = JCM 34227 t ) , be isolate from the soda lake Hutong Qagan Lake in Inner Mongolia Autonomous Region , China . the genomic size of the type strain be 2.57   mb , and dna g+c content be 49.7 mol% ( determine from the genome sequence).,this study be support by the National Natural Science Foundation of China ( no . 91751201 ) and the Science and Technology Basic Resources Investigation Project ( no . 2017fy100300).,the author declare that there be no conflict of interest . ,differentiate characteristic of strain IM 1326 T and the ten type strain of the genus AliidiomarinaStrains : 1 , IM 1326 T ( data from this study ) ; 2 , a. taiwanensis CGMCC 1.15255 T ; 3 , a. haloalkalitoleran DSM 24939 T ; 4 , a. sanyensis GYP-17 T ; 5 , a. shirensi AIST ; 6 , a. minuta MLST1 T ; 7 , a. iranensis GBPy7 T ; 8 , a. soli Y4G10 - 17 T ; 9 , a. maris CF12 - 14 T ; 10 , a. celeris F3105 T ; 11 , a. sedimenti GBSy1T. + , positive ; − , negative ; w , weakly positive ; nd , not detect ; DPG , diphosphatidylglycerol ; PG , phosphatidylglycerol ; PE : phosphatidylethanolamine ; APL , unidentified aminophospholipid ; PL , unidentified phospholipid . dna g+c content of strain IM 1326 T be obtain from the genomic sequence . datum for A. taiwanensis CGMCC 1.15255 t , and a. haloalkalitoleran DSM 24939 T be take from Srinivas et al . [ 2 ] , and some experiment be repeat under identical condition ( mark by * ) . datum for other specie be take from the literature : A. sanyensis GYP-17 T [ 6 ] , A. shirensi AIST [ 3 ] , A. minuta MLST1 T [ 4 ] , A. iranensis GBPy7 T [ 5 ] , a. soli Y4G10 - 17 T [ 7 ] , a. maris CF12 - 14 T [ 8 ] , a. celeris F3105 T [ 9 ] , and a. sedimenti GBSy1 T [ 10].,genome relatedness base on average nucleotide identity ( ANI ) , average amino acid identity ( aai ) , and digital dna – dna hybridization ( dddh ) value among strain IM 1326 T and specie of the genus Aliidiomarina . strain : 1 , IM 1326 t ; 2 , a. taiwanensis CGMCC 1.15255 t ; 3 , a. haloalkalitoleran DSM 24939 T ; 4 , a. sanyensis GYP-17 T ; 5 , a. AIST ; 6 , a. minuta MLST1 T ; 7 , a. iranensis GBPy7 T ; 8 , a. soli Y4G10 - 17 T ; 9 , a. maris CF12 - 14 T ; 10 , a. celeris F3105 T ; 11 , a. sedimenti GBSy1T. ANIb and ANIm value be calculate by blast and MUMmer algorithm , respectively . AAI value be calculate by ANI / AAI - Matrix . dDDH value be obtain by calculate genome - to - genome distance . nan nan ANI , average nucleotide identity ; dddh , digital dna – dna hybridization ; pha , poly - β - hydroxyalkanoate . ,the genus Mameliella be propose by Zheng et al . in 2010 , with Mameliella alba as the type species [ 1 ] . in 2021 , the ' Roseobacter clade ' of the family Rhodobacteraceae be validly publish as the family Roseobacteraceae [ 2 , 3 ] , therefore the genus Mameliella be move to the family Roseobacteraceae of the class Alphaproteobacteria . in 2015 , four heterotypic synonym of Mameliella alba , mameliella phaeodactyli [ 4 ] , mameliella atlantica [ 5 ] , ponticoccus lacteus [ 6 ] and alkalimicrobium pacificum [ 7 ] , have be publish independently , because the chimaeric 16s rrna gene sequence of the type strain JLT354 - WT ( eu734592.1 ) have be submit to the GenBank database [ 8 ] . after identification of this mistake , by January 2022 , the genus Mameliella contain only one species with a correct validly publish name accord to List of prokaryotic name with Standing in Nomenclature ( https://lpsn.dsmz.de/genus/mameliella).,during a course of study on the poly - β - hydroxyalkanoate - produce marine bacteria from Daya Bay , Guangdong , PR China , we isolate a novel bacterial strain resemble member of the genus Mameliella , designate DP3N28 - 2T. use PCR with specific primer [ 9 ] , we identify that DP3N28 - 2 T have a poly - β - hydroxyalkanoate ( pha ) polymerase ( phac ) gene ( gel profile datum not show ) , which indicate the potential for produce PHA . therefore , in this study , we be aim to determine the exact taxonomic position of DP3N28 - 2 T , by use a polyphasic approach base on phenotypic , chemotaxonomic and genotypic datum , and estimate the pha - produce ability of the genus Mameliella . ,the novel isolate of a member of the genus Mameliella be isolate from a surface sediment sample near off - shore sewage outfall ( 22 ° 34′19.57″ n , 114 ° 30′29.33″ e ) from 5.3   m depth . the sediment sample be incubate in medium contain 5   g ( w / w ) peptone and 1   g ( w / w ) yeast extract per litre sea water , at 30   ° c for 28   day , and then isolate use the standard dilution plating technique on marine agar ( MA ; Haibo ) . after purification by re - streaking , the novel strain be routinely cultivate at 30   ° c on MA and store at −80   ° c in sterile 15   % ( v / v ) glycerol supplement with 1   % ( w / v ) saline . the reference type strain , M. alba CGMCC 1.7209 T , be procure from the China General Microbiological Culture Collection Centre . both strain could form single colony on MA within 24   h at 30   ° c . to standardize the culture condition in this study , we cultivate M. alba CGMCC 1.7209 T use MA or MB.,The 16s rrna gene of DP3N28 - 2 T be amplify by pcr with the universal primer 27f and 1492r [ 10 ] , and sequence by Sangon ( Shanghai , PR China ) . the 16s rrna gene sequence of DP3N28 - 2 T be analyse by use the blast programme ( NCBI ) and compare with closely relate sequence of reference organism use the service from EzBioCloud ( www.ezbiocloud . net ) [ 11 ] . alignment of 16s rrna gene sequence be perform use the programme clustal_x , version 1.81 [ 12 ] , and position with insertion or deletion be exclude during calculation . phylogenetic tree be reconstruct by the neighbor - joining ( NJ ) [ 13 ] , maximum - parsimony ( mp ) [ 14 ] and maximum - likelihood ( ml ) [ 15 ] method with the mega 7 programme package [ 16 ] . the NJ tree and ML tree be reconstruct use evolutionary distance that be calculate with the Kimura two - parameter model with Gamma distribution [ 17 ] . the MP tree be obtain use the subtree - prune – regrafting algorithm [ 14 ] . all position contain gap and miss datum be eliminate . the stability of the cluster be evaluate by bootstrap analysis base on 1000 replicate . a nearly complete 16s rrna gene sequence of dp3n28 - 2 T obtain by amplification ( 1303   bp ) be include in the 16s rrna gene sequence assemble from genomic sequence ( 1437   bp ) . on the basis of the 16s rrna gene sequence ( from genome sequence ) , strain DP3N28 - 2 T be closely relate to Mameliella alba DSM 23384 T ( 98.3 % ) , Antarctobacter jejuensis 13–2 - B6 T ( 97.2 % ) , Antarctobacter heliothermus El-219 T ( 96.8 % ) , Maliponia aquimaris MM-10 T ( 96.7 % ) , Ponticoccus litoralis CL - GR66 T ( 96.4 % ) and Aquicoccus porphyridii L1 8 - 17 t ( 96.1 % ) . in the NJ , MP and ML phylogenetic tree ( fig . 1 ) , although the distance between DP3N28 - 2 T and M. alba be the short , they do not cluster together . therefore , a genome - base phylogenetic tree be reconstruct use the Type ( strain ) Genome Server provide by DSMZ [ 18 ] to far verify the phylogenetic position . this tree indicate that DP3N28 - 2 T form a clade with the type strain of M. alba ( fig . 2 ) , which indicate that DP3N28 - 2 T could represent a novel species of the genus Mameliella . ,,,,,the draft genome of DP3N28 - 2 T be sequence use a HiSeq X - ten platform ( Illumina ) , assemble at Guangdong Magigene Biotechnology . ( Guangdong , PR China ; http://www.magigene.com/ ) use the SPAdes software ( v3.13.0 ) [ 19 ] and annotate use the NCBI Prokaryotic Genome Annotation Pipeline [ 20 ] . the assemble genome of DP3N28 - 2 T ( JAHUZG000000000 ) have a total length of 5   120   735   bp , coverage of 600× , contig n50 value of 1   507   360   bp , 4966 predict gene sequence and 54 rna . the final assembly contain 21 contig and a have a dna g+c content of 63.0 mol%.,average nucleotide identity ( ANI ) use the OrthoANI algorithm [ 21 ] and digital dna – dna hybridization ( dddh ) value [ 22 ] be obtain to determine the genetic relatedness between DP3N28 - 2 T and the type species M. alba CGMCC 1.7290 T ( FMZI01000000 ) . ANI and dddh ( formula 2 ) be 81.1 and 23.4   % , respectively . they be both low than the threshold value for specie demarcation ( ani , 95–96   % ; dddh , 70   % ) . furthermore , the difference in dna g+c content between DP3N28 - 2 T and the type specie be 2.2 mol% , which indicate that they represent two different specie [ 23 ] . therefore , on the basis of the result of the phylogenetic analysis , the 16s rrna gene sequence similarity value and the genomic sequence similarity value , dp3n28 - 2 T be consider to represent a novel species of the genus Mameliella . ,all morphological and physiological feature of DP3N28 - 2 T be examine after incubation in MB or on ma at 30   ° c for 24   h to a maximum of 1   week unless otherwise specify . cell morphology , Gram - staining and motility be determine by microscopy ( CX43 , Olympus ) and transmission electron microscopy ( Tecnai G2 F30 , FEI ) . cell of DP3N28 - 2 T be gram - stain - negative non - flagellated and rod shape . the cell length be 1.3–4.7   µm and the width be 0.5–0.9   µm ( fig . s1 , available in the online version of this article ) . the shape vary due to the distribution of the PHA granule . growth of DP3N28 - 2 T and M. alba CGMCC 1.7290 T be test at 4 , 8 , 15 , 20 , 25 , 30 , 35 , 37 , 40 , 45 and 50   ° c , NaCl concentration in the range of 0–10   % ( w / v ; at increment of 1   % ) and ph value in the range of 4.0–12.0 ( at interval of 1.0   ph unit ) adjust use HCl or NaOH . the growth be measure use OD600 after 24 , 48 and 96   h. growth occur at temperature from 15 to 45   ° c ( optimally at 37–40   ° c ) , ph from 5.0 to 8.0 ( optimally at ph 6.0 ) and NaCl concentration from 0–10   % ( optimally at 4   % ) . no growth observe after incubation in an anaerobic chamber on MA for 2   week . oxidase activity be determine use a bioMérieux oxidase reagent kit accord to the manufacturer ’s instruction . catalase activity be test by bubble production use 3   % ( v / v ) h2o2 solution . utilization of citrate , pyruvate and malonate as sole carbon source be determine . DP3N28 - 2 T be cultivate in the minimal salt medium contain , per litre of distil water : 2.28   g k2hpo4·3h2o , 1.32   g ( nh4)2so4 , 0.47   g nahpo4 , supplement with 1   % carbon source . hydrolysis of agar and starch , h2s production , reduction of nitrate , Voges – Proskauer test , methyl red test and indole production be investigate accord to the method describe by Dong and Cai [ 24 ] . the phenotypic characterization be carry out use API 20NE , API ZYM ( bioMérieux ) and GEN III MicroPlate ( Biolog ) on DP3N28 - 2 T and the reference type strain independently twice without replicate at 30   ° c . the polyhydroxyalkanoate production of DP3N28 - 2 T be examine as describe by Juengert et al . [ 25 ] use gas chromatography ( GC-2014 , Shimadzu ) with GC column DB - WAX ( G6501 - CTC , Agilent Technologies).,DP3N28 - 2 T exhibit phenotypic similarity to M. alba CGMCC 1.7290 T , include be gram - stain - negative , rod shaped , aerobic , have positive reaction for catalase and oxidase and identical reaction in API ZYM test , except for α - galactosidase . both strain could accumulate poly - β - hydroxybutyrate . the GC result indicate that DP3N28 - 2 t and m. alba cgmcc 1.7290 t accumulate pha up to 12.3±2.9   %   and 19.2±5.8   % ( per dry cell weight , ±standard deviation ) , respectively . in the genomic datum , the PHA synthase gene ( phaC ) , PHA depolymerase gene ( phaZ ) , regulator gene ( phar ) and PHA granule - associate phasin gene ( phap ) be annotate ( accession number be list in Table S1).,On the basis of the datum obtain during this study , we could distinguish DP3N28 - 2 T from M. alba in various way ( Table 1 ) . however , the report phenotypic feature of M. alba be highly variable among the five representative [ 8 ] , for example the optimal nacl ( w / v , % ) of M. alba JLT354 - WT be 1–3 [ 1 ] while that of A. pacificum F15 T be 3.5 [ 7 ] , from which we could draw different conclusion about the similarity between strain DP3N28 - 2 T and M. alba . therefore , we identify three feature include be slightly curved , negative for nitrate reduction and prefer acidic growth condition ( ph 6.0 ) as the critical phenotypic difference between DP3N28 - 2 T and M. alba . the detailed physiological and biochemical characteristic of DP3N28 - 2 T be give in table 1 and 2 , table s2 and the specie description . accord to the Biolog GEN III MicroPlate test , DP3N28 - 2 T be susceptible to troleandomycin , nalidixic acid , aztreonam and niaproof 4 , and resistant to lincomycin , vancomycin , fusidic acid , rifamycin sv , guanidine HCl and minocycline . culture of DP3N28 - 2 T and M. alba CGMCC 1.7290 t be grow in MB at 30   ° c for 48   h , and then freeze - dry for polar lipid , respiratory quinone and cellular fatty acid composition analysis . the cellular fatty acid composition be measure by the Marine Culture Collection of China ( MCCC ; Xiamen , Fujian , China ) use the Microbial ID system [ 26 ] . respiratory quinone of DP3N28 - 2 T be determine by the MCCC use ultra - performance liquid chromatography – tandem mass spectrometry method [ 27 ] . the cellular polar lipid be extract and determine by two - dimensional thin - layer chromatography use silica gel plate ( 60   f254 , Merck ) [ 28 , 29 ] . total lipid material be detect use molybdatophosphoric acid , and specific functional group be detect use ninhydrin and phosphomolybdenum blue reagent [ 29 , 30 ] . the respiratory quinone of both strain be q-10 . as show in table 2 , the two strain show similar cellular fatty acid profile , with sum feature 8 , c16   :   0 and c18   :   0 as the top three component ( & gt;5   % ) , however , in M. alba CGMCC 1.7290 T , the proportion of c18   :   0 be high than that of c16   :   0 , which be consistently show in all the five representative of M. alba [ 8 ] . as show in fig . s2 , the polar lipid of DP3N28 - 2 T include phosphatidylethanolamine , phosphatidylmonomethylethanolamine , phosphatidylglycerol , one aminophospholipid , one phospholipid and three unidentified lipid . the polar lipid composition be variable among the M. alba strain [ 8 ] , but none of they contain phosphatidylmonomethylethanolamine , which could be use to distinguish DP3N28 - 2 T from M. alba . ,On the basis of these phenotypic , chemotaxonomic , phylogenetic and genetic datum , we propose to classify strain DP3N28 - 2 T as represent a novel species of the genus Mameliella , for which the name Mameliella sediminis sp . nov . be propose . ,Mameliella sediminis ( se.di’mi.nis l. gen . neut . n. sediminis , of sediment).,cell be gram - stain - negative , aerobic , and non - motile rod ( 0.5–0.9   µm wide and 1.3–4.7   µm long ) . small colony ( 1–2   mm ) be produce on MA after incubation at 30   ° c for 24   h. colony be smooth , convex and beige . growth occur at 15–45   ° c , at ph 5.0–8.0 and in the presence of 0–10   %   nacl ; optimal growth occur at 37–40   ° c , at ph 6.0 and in the presence of 4.0   % nacl . positive for catalase and oxidase . negative for nitrate reduction , hydrolysis of gelatin and starch , methyl red test , Voges – Proskauer reaction , and h2s production . pyruvate be use as sole carbon source , but not citrate . in the API ZYM test , positive for alkaline phosphatase , esterase ( C4 ) , esterase lipase ( C8 ) , lipase ( c14 ) , leucine arylamidase , valine arylamidase , cystine arylamidase , acid phosphatase , naphthol - as - bi - phosphohydrolase and α - galactosidase ; negative for trypsin , α - chymotrypsin , β - galactosidase , β - glucuronidase , α - glucosidase , β - glucosidase , n - acetyl - β - glucosaminidase , α - mannosidase and α - fucosidase . in the API 20ne test , positive for d - glucose fermentation , urease , β - glucosidase ( aesculin hydrolysis ) and β - galactosidase ; weakly positive for gelatin hydrolysis , utilization of d - glucose , l - arabinose , and d - mannose ; negative for reduction of nitrate to nitrite , denitrification , indole production , arginine dihydrolase , utilization of d - mannitol , n - acetyl - glucosamine , maltose , potassium gluconate , capric acid , malic acid , adipic acid , phenylacetic acid and trisodium citrate . accord to Biolog GEN III MicroPlate assay , α - hydroxybutyric acid , l - arginine , l - lactic acid , β - hydroxy - d , l - butyric acid , d - fucose , acetoacetic acid and trehalose be oxidize . the polar lipid be phosphatidylethanolamine , phosphatidylmonomethylethanolamine , phosphatidylglycerol , one aminophosphlipid , one phospholipid and three unidentified lipid . the main fatty acid be sum feature 8 ( c18   :   1ω6c and/or c18   :   1ω7c ) and c16   :   0 . the respiratory quinone be q-10.,the type strain , DP3N28 - 2 T (= MCCC 1k06218t = KCTC 82804 t ) be isolate from the sediment collect from Daya Bay , Guangdong , PR China . the dna g+c content of the type strain be 63.0 mol   % ( datum from genome sequence ) . the GenBank accession number for the 16s rrna gene sequence and draft genome of strain dp3n28 - 2 t be mz277401 and jahuzg000000000 respectively . ,this work be support by the Key Program of Marine Economy Development ( six Marine Industries ) Special Foundation of Department of Natural Resources of Guangdong Province ( GDNRC[2020]034 ) and China Postdoctoral Science Foundation ( 2019M662806).,W.S.Z. , S.Q.Z. and Y.H. design the study ; W.S.Z. , S.Q.Z. and M.J.Z. perform the experiment ; W.S.Z. and S.Q.Z. analyse the datum ; Y.H. and W.S.Z. support the research ; W.S.Z. , S.Q.Z. and Y.H. write the paper . ,the author declare that there be no conflict of interest . ,characteristic of DP3N28 - 2 T and the type species of the genus Mameliella . strain : 1 , Mameliella sp . DP3N28−2 T ; 2 , M. alba CGMCC 1.7290t. all datum be obtain from this study unless otherwise indicate . + , positive or grow ; − , negative or not grow ; W , weakly positive or grow . the detailed datum for API ZYM , API 20NE and Biolog GEN III be give in Table s2.,cellular fatty acid content ( percentage ) of DP3N28 - 2 T and Mameliella albastrain : 1 , DP3N28 - 2 t ; 2 , M. alba CGMCC 1.7290t. all datum be obtain during this study . major fatty acid ( & gt;10 % ) be highlight in bold type . only fatty acid exceed 1.0   % of the total cellular fatty acid of at least one of the strain be show . tr , Trace ( & lt;1.0 % ) ; nd , not detect . nan\n"
     ]
    }
   ],
   "source": [
    "lemmatized_articles = \" \".join([token.lemma_ for token in doc])\n",
    " \n",
    "print(lemmatized_articles)\n",
    "# https://www.geeksforgeeks.org/python-lemmatization-approaches-with-examples/"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "558bbc1e",
   "metadata": {},
   "source": [
    "## Lemmatizing of specific df column"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "2a4361fa",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0        a haloalkaliphilic strain ( IM 1326 T ) be iso...\n",
      "1        through bacterial plant – endophyte extraction...\n",
      "2        a gram - negative , moderately halophilic bact...\n",
      "3        a gram - stain - negative , strictly aerobic ,...\n",
      "4        a strictly anaerobic bacterial strain , design...\n",
      "                               ...                        \n",
      "17001                                                  NaN\n",
      "17002                                                  NaN\n",
      "17003                                                  NaN\n",
      "17004                                                  NaN\n",
      "17005                                                  NaN\n",
      "Name: abstract_lemma, Length: 17006, dtype: object\n"
     ]
    }
   ],
   "source": [
    "df[\"abstract_lemma\"] = df['abstract'].head(5).astype(str).apply(lambda x: \" \".join([y.lemma_ for y in nlp(x)]))\n",
    "print(df[\"abstract_lemma\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "152e4910",
   "metadata": {},
   "source": [
    "## Sentences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "160056c5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This genus has two sister genera named Idiomarina and Pseudidiomarina in the List of Prokaryotic names with Standing in Nomenclature.\n"
     ]
    }
   ],
   "source": [
    "#save sentences of text in list, so one can iterate over the sentences\n",
    "sentences = list(doc.sents)\n",
    "print(sentences[3])"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4516203d",
   "metadata": {},
   "source": [
    "## Named Entity Recognotion (NER)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "49c03bbd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[two, 78, ten, 5, 6, 7, 4, 5, 6, Hutong Qagan Lake, the Ordos, Inner Mongolia Autonomous Region, China, July 2017, 39° 12′ 14″ N, 109° 0′ 8″ E, 1250 , 26%, 9.95, 37 , 2 , 15.0 , 0.5 , 6.0 , 4.0 , 0.2 , 2.0 , 0.25 , 0.38 , 0.25 , 0.25 , 1.0 , 0.08 , 56 , 9.5, 1 M NaOH, 15.0 , 37 , 5 , 5 , 37 , 37 , 20 , −80 , Takara, Witon Information Technology Co., Ltd., Beijing, China, 1000, 1503 , 95.8, 95.7, 95.4, 95.3, 95.0, 94.7, 93.6%, IM 1326, 1, less than 70%, 1, lower than, 70 , Biomarker Technologies, Beijing, PR China, Qubit, Invitrogen, Illumina, NovaSeq, 3.13.0, NCBI, the JSpecies Web Server, Meier-Kolthoff et al, 120, the Genome Taxonomy Database, IM 1326, 3.06 , 19, 12, 2.57 , IM 1326, 49.7 mol%, 46.3–52.1 , 48.7 mol%, 1, 68.1 to, 72.6 , 80.7 to, 84.9 , 76 to, 78 , 16.2 to, 22.7 , 2, 72.6 , 83.5 , 78 , 18.4 , 95 , 95 , 70 , 120, 2, 100%, 120, 78, the China General Microbiological Culture Collection Center, the Leibniz Institute German Collection of Microorganisms and Cell Cultures, 5.0 , 1.0 , 0.1 , 19.45 , 5.98 , 3.24 , 1.8 , 0.55 , 0.16 , 0.08 , 34.0 , 22.0 , 4.0 , 2.4 , 1.6 , 8.0 , 30 , 30.0, 2.0, 1.0, 0.23, 0.27, 0.06, 5.0, 10.0, 15.0, 37 , 8.5, Dussault, 20, Hitachi, Olympus, BX 51, 1, 4, 15, 20, 25, 30, 34, 37, 40, 42, 45, 50 , 3 , 9.0, 1.0 , 4 , 5.0, 12.0, 0.5, 25, 5.0 , 3.0 , 5.0 , 20, 40, 60, 80, 4 , Harrigan, 21, 22, 23, Oren, 24, 1 , 10.0 , approximately 0.3–0.6 , 0.8–2.1 , S9, 8.5, 24 , 37 , 0–13.0 , 4–42 , 37 , IM 1326, 20, 40, four, IM 1326, 2 , 37 , 6.0, IM 1326, 9, 22.2 , 16.1 , 13.1 , two, up to 13.2 , two, three, first, second, 65 , 80 , two, 1, Collins, IM 1326, 1, approximately 0.3–0.6 , 0.8–2.1 , 24 , 37, approximately 1.0–3.0 , 4–45 , 37 , 0–13.0 , 20, 40, 60, 80, 9, one, Hutong Qagan Lake, Inner Mongolia Autonomous Region, China, 2.57 , 49.7 mol%, the National Natural Science Foundation of China, the Science and Technology Basic Resources Investigation Project, ten, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, Srinivas, 2, 6, 3, 4, 5, 7, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, Zheng, 2010, 2021, 2015, four, 6, 7, GenBank, January 2022, only one, Daya Bay, Guangdong, China, DP3N28-2, 22°34′19.57″ N, 114°30′29.33″ E, 5.3 , 5, 1, 30 , 28 days, 30 , −80 , 15 , 1 , the China General Microbiological Culture Collection Centre, 24 , 30 , Sangon, Shanghai, China, two, 1000, 1303 , 1437 , 98.3%, 97.2%, 96.8%, 96.7%, 96.4%, 96.1%, 1, the Type (Strain) Genome Server, DSMZ, 2, HiSeq X-ten, Illumina, Guangdong Magigene Biotechnology, Guangdong, China, 5 , 735 , 600×, 1 , 507 , 360 , 4966, 54, 21, 63.0 mol%.,Average nucleotide identity (ANI) using the OrthoANI algorithm [21] and digital DNA–DNA hybridization (dDDH) values [22] was obtained to determine the genetic relatedness between DP3N28-2T and the type species M. alba CGMCC 1.7290T (FMZI01000000)., 2, 81.1 and, 23.4 , 95–96 , 70 , 2.2 mol%, which indicated that they represented two different species [23]., 30 , 24 , CX43, Olympus, Tecnai G2 F30, FEI, 1.3–4.7 , 0.5–0.9 , 4, 8, 15, 20, 25, 30, 35, 37, 40, 45, 50 , 0–10 , 1 , 1.0 , 24, 48, 96 , 15 to 45 , 37–40 , 5.0, 8.0, 6.0, 0–10 , 4 , 2 , weeks, 3 , 2.28 , 1.32 , 0.47 , 1 , Voges, Dong, Cai, 24, 30 , Juengert, Shimadzu, Agilent Technologies).,DP3N28, up to, 12.3±2.9 , 19.2±5.8 , 1, five, 3.5, three, 6.0, 1, 2, 30 , 48 , the Marine Culture Collection of China, MCCC, Xiamen, Fujian, China, Microbial ID system, 26, MCCC, two, 60 , Merck, 28, 29, 29, 30, 2, two, three, five, one, one, three, DP3N28-2, 1.3–4.7 , 1–2 , 30 , 24 , 15–45 , 0–10 , 37–40 , 6.0, 4.0 , one, one, three, 8, Daya Bay, Guangdong, China, 63.0 mol , GenBank, the Key Program of Marine Economy Development (Six Marine Industries) Special Foundation of Department of Natural Resources of Guangdong Province, China Postdoctoral Science Foundation, W.S.Z., W.S.Z., Y.H., W.S.Z., 1, 2, 1, 2, 1.0 , at least one]\n"
     ]
    }
   ],
   "source": [
    "# save entities of a sentece in a list\n",
    "sentence = sentences[42]\n",
    "ents = list(doc.ents)\n",
    "print(ents)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 268,
   "id": "cd832ba7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[1900, Tissier, Two, 17:00, 8th, Bergey, 8th, Bergey, four]\n"
     ]
    }
   ],
   "source": [
    "\"\"\" GOAL: List of sentences, which contains species and keywords\n",
    "if ent\"SPECIES\" and ent\"KEYWORD in sentences:\n",
    "    print sentence\n",
    "\"\"\""
   ]
  },
  {
   "cell_type": "markdown",
   "id": "46c69ed6",
   "metadata": {},
   "source": [
    "## POS (Part of Speech) tagging"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "d5a83a82",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This DET det\n",
      "genus NOUN nsubj\n",
      "has VERB ROOT\n",
      "two NUM nummod\n",
      "sister NOUN compound\n",
      "genera NOUN dobj\n",
      "named VERB acl\n",
      "Idiomarina PROPN oprd\n",
      "and CCONJ cc\n",
      "Pseudidiomarina PROPN conj\n",
      "in ADP prep\n",
      "the DET det\n",
      "List PROPN pobj\n",
      "of ADP prep\n",
      "Prokaryotic ADJ amod\n",
      "names NOUN pobj\n",
      "with ADP prep\n",
      "Standing PROPN pcomp\n",
      "in ADP prep\n",
      "Nomenclature PROPN pobj\n",
      ". PUNCT punct\n"
     ]
    }
   ],
   "source": [
    "# outputs every POS for every token in the sentence\n",
    "for token in sentences[3]:\n",
    "    print(token.text, token.pos_, token.dep_)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e669a43c",
   "metadata": {},
   "source": [
    "## Extracting noun chunks"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "8caa176f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "the species\n",
      "ten species\n",
      "These species\n",
      "a novel species\n",
      "the most closely related species\n",
      "the type species\n",
      "species\n",
      "a novel species\n",
      "other species\n",
      "other species\n",
      "the type species\n",
      "its closely related species\n",
      "the most closely related species\n",
      "species delineation\n",
      "other species\n",
      "a novel species\n",
      "a novel species\n",
      "type species\n",
      "reference species\n",
      "the reference species\n",
      "the reference species\n",
      "the species description\n",
      "two reference species\n",
      "all three species\n",
      "two reference species\n",
      "the type species\n",
      "the Aliidiomarina species\n",
      "a novel species\n",
      "other species\n",
      "species\n",
      "the type species\n",
      "only one species\n",
      "a novel species\n",
      "the type species\n",
      "species demarcation\n",
      "the type species\n",
      "two different species\n",
      "a novel species\n",
      "the species descriptions\n",
      "a novel species\n",
      "the type species\n"
     ]
    }
   ],
   "source": [
    "# finds the noun \"species\" with additional word\n",
    "chunks = list(doc.noun_chunks)\n",
    "for chunk in chunks:\n",
    "    if \"species\" in str(chunk):\n",
    "                       print(chunk)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e29fbfd3",
   "metadata": {},
   "source": [
    "## Extracting verbs and verb phrases"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "15f4e705",
   "metadata": {},
   "outputs": [],
   "source": [
    "import textacy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "db4380be",
   "metadata": {},
   "outputs": [],
   "source": [
    "# define a pattern to search for in the document \n",
    "patterns = [{\"POS\": \"ADV\"}, {\"POS\": \"VERB\"}]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "cb816c45",
   "metadata": {},
   "outputs": [],
   "source": [
    "verb_phrases = textacy.extract.token_matches(doc, patterns=patterns)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "e5aa22fb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "validly published\n",
      "originally designed\n",
      "routinely cultured\n",
      "preliminarily analysed\n",
      "also supported\n",
      "closely related\n",
      "then comparing\n",
      "validly published\n",
      "validly published\n",
      "then isolated\n",
      "routinely cultivated\n",
      "closely related\n",
      "closely related\n",
      "further verify\n",
      "otherwise specified\n",
      "consistently shown\n",
      "otherwise indicated\n"
     ]
    }
   ],
   "source": [
    "for verb_phrase in verb_phrases:\n",
    "    print(verb_phrase)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "11da24d3",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1baa8011",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "venv_spacy",
   "language": "python",
   "name": "venv_spacy"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
